diff -urN ../resources-original/geoserver/sld/StyledLayerDescriptor.xsd geoserver/sld/StyledLayerDescriptor.xsd
--- ../resources-original/geoserver/sld/StyledLayerDescriptor.xsd	2019-02-07 18:38:11.685509200 +0100
+++ geoserver/sld/StyledLayerDescriptor.xsd	2019-02-07 18:33:40.507954500 +0100
@@ -6,9 +6,9 @@
             xmlns:xsd="http://www.w3.org/2001/XMLSchema"
             elementFormDefault="qualified">
   <xsd:import namespace="http://www.w3.org/1999/xlink"
-              schemaLocation="../xlink/1.0.0/xlinks.xsd"/>
+              schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
   <xsd:import namespace="http://www.opengis.net/ogc"
-              schemaLocation="../filter/1.0.0/filter.xsd"/>
+              schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
 
 <!-- *********************************************************************** -->
   <xsd:annotation>
@@ -146,7 +146,7 @@
       </xsd:documentation>
     </xsd:annotation>
     <xsd:complexType>
-      <xsd:attributeGroup ref="xlink:simpleLink"/>
+      <xsd:attributeGroup ref="xlink:simpleAttrs"/>
     </xsd:complexType>
   </xsd:element>
 
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsCommon.xsd geoserver/wcs/1.1.1/wcsCommon.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsCommon.xsd	2019-02-07 18:01:17.982003500 +0100
+++ geoserver/wcs/1.1.1/wcsCommon.xsd	2019-02-07 18:33:40.462800500 +0100
@@ -15,7 +15,7 @@
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================		
 	elements and types	
 	============================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsContents.xsd geoserver/wcs/1.1.1/wcsContents.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsContents.xsd	2019-02-07 18:01:17.991022400 +0100
+++ geoserver/wcs/1.1.1/wcsContents.xsd	2019-02-07 18:33:40.470803700 +0100
@@ -13,7 +13,7 @@
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsCommon.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsCoverages.xsd geoserver/wcs/1.1.1/wcsCoverages.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsCoverages.xsd	2019-02-07 18:27:03.044189800 +0100
+++ geoserver/wcs/1.1.1/wcsCoverages.xsd	2019-02-07 18:33:40.487341200 +0100
@@ -12,7 +12,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
 	<!-- ==========================================================
 		Types and elements
 	    ========================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsDescribeCoverage.xsd geoserver/wcs/1.1.1/wcsDescribeCoverage.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsDescribeCoverage.xsd	2019-02-07 18:08:29.372641100 +0100
+++ geoserver/wcs/1.1.1/wcsDescribeCoverage.xsd	2019-02-07 18:33:40.477828200 +0100
@@ -19,8 +19,8 @@
 	<include schemaLocation="wcsCommon.xsd"/>
 	<include schemaLocation="wcsGridCRS.xsd"/>
 	<include schemaLocation="wcsInterpolationMethod.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsGetCapabilities.xsd geoserver/wcs/1.1.1/wcsGetCapabilities.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsGetCapabilities.xsd	2019-02-07 18:08:44.925854500 +0100
+++ geoserver/wcs/1.1.1/wcsGetCapabilities.xsd	2019-02-07 18:33:40.480805200 +0100
@@ -13,7 +13,7 @@
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsContents.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsGetCoverage.xsd geoserver/wcs/1.1.1/wcsGetCoverage.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsGetCoverage.xsd	2019-02-07 18:09:04.180507000 +0100
+++ geoserver/wcs/1.1.1/wcsGetCoverage.xsd	2019-02-07 18:33:40.494548500 +0100
@@ -19,8 +19,8 @@
 	<include schemaLocation="wcsCommon.xsd"/>
 	<include schemaLocation="wcsGridCRS.xsd"/>
 	<include schemaLocation="wcsCoverages.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================	
 		elements and types	
 	============================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsGridCRS.xsd geoserver/wcs/1.1.1/wcsGridCRS.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsGridCRS.xsd	2019-02-07 18:01:18.075436000 +0100
+++ geoserver/wcs/1.1.1/wcsGridCRS.xsd	2019-02-07 18:33:40.501486700 +0100
@@ -16,7 +16,7 @@
 	<!-- ========================================================
            includes and imports
 	======================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<!-- ========================================================
 		elements and types
 	======================================================== -->
diff -urN ../resources-original/geoserver/wcs/1.1.1/wcsInterpolationMethod.xsd geoserver/wcs/1.1.1/wcsInterpolationMethod.xsd
--- ../resources-original/geoserver/wcs/1.1.1/wcsInterpolationMethod.xsd	2019-02-07 18:35:32.594671500 +0100
+++ geoserver/wcs/1.1.1/wcsInterpolationMethod.xsd	2019-02-07 18:37:00.891891400 +0100
@@ -12,7 +12,7 @@
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
