diff -urN ../resources-original/geoserver/sld/StyledLayerDescriptor.xsd geoserver/sld/StyledLayerDescriptor.xsd
--- ../resources-original/geoserver/sld/StyledLayerDescriptor.xsd	Wed Apr 15 22:05:57 2015
+++ geoserver/sld/StyledLayerDescriptor.xsd	Sun Jul 10 18:00:57 2016
@@ -6,9 +6,9 @@
             xmlns:xsd="http://www.w3.org/2001/XMLSchema"
             elementFormDefault="qualified">
   <xsd:import namespace="http://www.w3.org/1999/xlink"
-              schemaLocation="../xlink/1.0.0/xlinks.xsd"/>
+              schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
   <xsd:import namespace="http://www.opengis.net/ogc"
-              schemaLocation="../filter/1.0.0/filter.xsd"/>
+              schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
 
 <!-- *********************************************************************** -->
   <xsd:annotation>
@@ -146,7 +146,7 @@
       </xsd:documentation>
     </xsd:annotation>
     <xsd:complexType>
-      <xsd:attributeGroup ref="xlink:simpleLink"/>
+      <xsd:attributeGroup ref="xlink:simpleAttrs"/>
     </xsd:complexType>
   </xsd:element>
 
