diff -urN ../resources-original/isotc211/2005/gmi/acquisitionInformation.xsd isotc211/2005/gmi/acquisitionInformation.xsd
--- ../resources-original/isotc211/2005/gmi/acquisitionInformation.xsd	Wed Sep 23 08:51:07 2015
+++ isotc211/2005/gmi/acquisitionInformation.xsd	Sun Jul 10 18:00:57 2016
@@ -6,9 +6,9 @@
 		<xs:documentation>2012-12-08: Documentation of MI_Objective_Type. Typo reported by XMG: "temportal" replaced by "temporal".</xs:documentation>
 	</xs:annotation>
 	<!-- ================================== Imports ================================== -->
-	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gmd/gmd.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gco/gco.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gss" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gss/gss.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gco/gco.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gss" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gss/gss.xsd"/>
 	<!-- ########################################################################### -->
 	<!-- ########################################################################### -->
 	<!-- ================================== Classes ================================= -->
diff -urN ../resources-original/isotc211/2005/gmi/contentInformation.xsd isotc211/2005/gmi/contentInformation.xsd
--- ../resources-original/isotc211/2005/gmi/contentInformation.xsd	Wed Sep 23 08:51:30 2015
+++ isotc211/2005/gmi/contentInformation.xsd	Sun Jul 10 18:00:57 2016
@@ -5,8 +5,8 @@
 		<xs:documentation>This file was generated from ISO TC/211 UML class diagrams == 07-30-2010 11:34:35 ====== Name: Content - Position: 5</xs:documentation>
 	</xs:annotation>
 	<!-- ================================== Imports ================================== -->
-	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gco/gco.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gmd/gmd.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gco/gco.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
 	<!-- ########################################################################### -->
 	<!-- ########################################################################### -->
 	<!-- ================================== Classes ================================= -->
diff -urN ../resources-original/isotc211/2005/gmi/dataQualityInformation.xsd isotc211/2005/gmi/dataQualityInformation.xsd
--- ../resources-original/isotc211/2005/gmi/dataQualityInformation.xsd	Wed Sep 23 08:51:44 2015
+++ isotc211/2005/gmi/dataQualityInformation.xsd	Sun Jul 10 18:00:57 2016
@@ -6,8 +6,8 @@
 		<xs:documentation>2012-12-08: Documentation of LE_ProcessStepReport_Type. Typo reported by XMG: "occured" replaced by "occurred".</xs:documentation>
 	</xs:annotation>
 	<!-- ================================== Imports ================================== -->
-	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gmd/gmd.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gco/gco.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gco/gco.xsd"/>
 	<!-- ########################################################################### -->
 	<!-- ########################################################################### -->
 	<!-- ================================== Classes ================================= -->
diff -urN ../resources-original/isotc211/2005/gmi/metadataEntitySet.xsd isotc211/2005/gmi/metadataEntitySet.xsd
--- ../resources-original/isotc211/2005/gmi/metadataEntitySet.xsd	Wed Sep 23 08:52:11 2015
+++ isotc211/2005/gmi/metadataEntitySet.xsd	Sun Jul 10 18:00:57 2016
@@ -5,8 +5,8 @@
 		<xs:documentation>This file was generated from ISO TC/211 UML class diagrams == 07-30-2010 11:34:30 ====== </xs:documentation>
 	</xs:annotation>
 	<!-- ================================== Imports ================================== -->
-	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gmd/gmd.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gco/gco.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gco/gco.xsd"/>
 	<xs:include schemaLocation="../gmi/acquisitionInformation.xsd"/>
 	<!-- ########################################################################### -->
 	<!-- ########################################################################### -->
diff -urN ../resources-original/isotc211/2005/gmi/spatialRepresentationInformation.xsd isotc211/2005/gmi/spatialRepresentationInformation.xsd
--- ../resources-original/isotc211/2005/gmi/spatialRepresentationInformation.xsd	Wed Sep 23 08:52:18 2015
+++ isotc211/2005/gmi/spatialRepresentationInformation.xsd	Sun Jul 10 18:00:57 2016
@@ -6,9 +6,9 @@
 	</xs:annotation>
 	<!-- ================================== Imports ================================== -->
 	<xs:include schemaLocation="../gmi/acquisitionInformation.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gmd/gmd.xsd"/>
-	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/gco/gco.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml/3.2" schemaLocation="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19136_Schemas/gml.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<xs:import namespace="http://www.isotc211.org/2005/gco" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gco/gco.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml/3.2" schemaLocation="http://schemas.opengis.net/gml/3.2.1/geometryBasic0d1d.xsd"/>
 	<!-- ########################################################################### -->
 	<!-- ########################################################################### -->
 	<!-- ================================== Classes ================================= -->
