diff -urN ../resources-original/ogc/context/1.0.0/collection.xsd ogc/context/1.0.0/collection.xsd
--- ../resources-original/ogc/context/1.0.0/collection.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/context/1.0.0/collection.xsd	Sun Jul 10 18:00:57 2016
@@ -8,6 +8,7 @@
 -->
 <xs:schema targetNamespace="http://www.opengis.net/context" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:context="http://www.opengis.net/context" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" version="1.0.0.2">
 	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
+	<xs:include schemaLocation="context.xsd"/>
 	<xs:element name="ViewContextCollection" type="context:ViewContextCollectionType"/>
 	<xs:complexType name="ViewContextCollectionType">
 		<xs:sequence>
@@ -28,7 +29,9 @@
 			<xs:element name="OnlineResource" type="context:OnlineResourceType"/>
 		</xs:sequence>
 	</xs:complexType>
+	<!--
 	<xs:complexType name="OnlineResourceType">
 		<xs:attributeGroup ref="xlink:simpleAttrs"/>
 	</xs:complexType>
+	-->
 </xs:schema>
diff -urN ../resources-original/ogc/eop/1.0/eop.xsd ogc/eop/1.0/eop.xsd
--- ../resources-original/ogc/eop/1.0/eop.xsd	Tue Nov  1 19:17:42 2011
+++ ogc/eop/1.0/eop.xsd	Sun Jul 10 18:00:57 2016
@@ -9,7 +9,7 @@
 		</xs:documentation>
 	</xs:annotation>
 	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://schemas.opengis.net/xlink/1.0.0/xlinks.xsd"/>
+	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<xs:element name="EarthObservation" type="eop:EarthObservationType" substitutionGroup="gml:Observation">
 		<xs:annotation>
 			<xs:documentation>eop root element for generic Earth Observation Product description</xs:documentation>
diff -urN ../resources-original/ogc/filter/1.0.0/expr.xsd ogc/filter/1.0.0/expr.xsd
--- ../resources-original/ogc/filter/1.0.0/expr.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/filter/1.0.0/expr.xsd	Sun Jul 10 18:00:57 2016
@@ -68,7 +68,11 @@
   </xsd:complexType>
   <xsd:complexType name="PropertyNameType">
     <xsd:complexContent mixed="true">
-      <xsd:extension base="ogc:ExpressionType"/>
+      <xsd:extension base="ogc:ExpressionType">
+        <xsd:sequence>
+          <xsd:any minOccurs="0"/>
+        </xsd:sequence>
+      </xsd:extension>
     </xsd:complexContent>
   </xsd:complexType>
 </xsd:schema>
diff -urN ../resources-original/ogc/filter/1.0.0/filterCapabilities.xsd ogc/filter/1.0.0/filterCapabilities.xsd
--- ../resources-original/ogc/filter/1.0.0/filterCapabilities.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/filter/1.0.0/filterCapabilities.xsd	Sun Jul 10 18:00:57 2016
@@ -16,15 +16,19 @@
          <xsd:element name="Functions" type="ogc:FunctionsType"/>
       </xsd:choice>
    </xsd:complexType>
+   <!--
    <xsd:element name="BBOX">
       <xsd:complexType/>
    </xsd:element>
+   -->
    <xsd:element name="Between">
       <xsd:complexType/>
    </xsd:element>
+   <!--
    <xsd:element name="Beyond">
       <xsd:complexType/>
    </xsd:element>
+   -->
    <xsd:complexType name="Comparison_OperatorsType">
       <xsd:choice maxOccurs="unbounded">
          <xsd:element ref="ogc:Simple_Comparisons"/>
@@ -33,6 +37,7 @@
          <xsd:element ref="ogc:NullCheck"/>
       </xsd:choice>
    </xsd:complexType>
+   <!--
    <xsd:element name="Contains">
       <xsd:complexType/>
    </xsd:element>
@@ -48,6 +53,7 @@
    <xsd:element name="Equals">
       <xsd:complexType/>
    </xsd:element>
+   -->
    <xsd:element name="Filter_Capabilities">
       <xsd:complexType>
          <xsd:sequence>
@@ -75,9 +81,11 @@
          <xsd:element name="Function_Names" type="ogc:Function_NamesType"/>
       </xsd:sequence>
    </xsd:complexType>
+   <!--
    <xsd:element name="Intersect">
       <xsd:complexType/>
    </xsd:element>
+   -->
    <xsd:element name="Like">
       <xsd:complexType/>
    </xsd:element>
@@ -87,9 +95,11 @@
    <xsd:element name="NullCheck">
       <xsd:complexType/>
    </xsd:element>
+   <!--
    <xsd:element name="Overlaps">
       <xsd:complexType/>
    </xsd:element>
+   -->
    <xsd:complexType name="Scalar_CapabilitiesType">
       <xsd:choice maxOccurs="unbounded">
          <xsd:element ref="ogc:Logical_Operators"/>
@@ -113,23 +123,47 @@
    </xsd:complexType>
    <xsd:complexType name="Spatial_OperatorsType">
       <xsd:choice maxOccurs="unbounded">
-         <xsd:element ref="ogc:BBOX"/>
-         <xsd:element ref="ogc:Equals"/>
-         <xsd:element ref="ogc:Disjoint"/>
-         <xsd:element ref="ogc:Intersect"/>
-         <xsd:element ref="ogc:Touches"/>
-         <xsd:element ref="ogc:Crosses"/>
-         <xsd:element ref="ogc:Within"/>
-         <xsd:element ref="ogc:Contains"/>
-         <xsd:element ref="ogc:Overlaps"/>
-         <xsd:element ref="ogc:Beyond"/>
-         <xsd:element ref="ogc:DWithin"/>
+         <xsd:element name="BBOX">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Equals">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Disjoint">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Intersect">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Touches">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Crosses">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Within">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Contains">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Overlaps">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="Beyond">
+            <xsd:complexType/>
+         </xsd:element>
+         <xsd:element name="DWithin">
+            <xsd:complexType/>
+         </xsd:element>
       </xsd:choice>
    </xsd:complexType>
+   <!--
    <xsd:element name="Touches">
       <xsd:complexType/>
    </xsd:element>
    <xsd:element name="Within">
       <xsd:complexType/>
    </xsd:element>
+   -->
 </xsd:schema>
diff -urN ../resources-original/ogc/filter/1.1.0/expr.xsd ogc/filter/1.1.0/expr.xsd
--- ../resources-original/ogc/filter/1.1.0/expr.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/filter/1.1.0/expr.xsd	Sun Jul 10 18:00:58 2016
@@ -61,7 +61,11 @@
    </xsd:complexType>
    <xsd:complexType name="PropertyNameType">
       <xsd:complexContent mixed="true">
-         <xsd:extension base="ogc:ExpressionType"/>
+        <xsd:extension base="ogc:ExpressionType">
+          <xsd:sequence>
+            <xsd:any minOccurs="0"/>
+          </xsd:sequence>
+        </xsd:extension>
       </xsd:complexContent>
    </xsd:complexType>
 </xsd:schema>
diff -urN ../resources-original/ogc/filter/1.1.0/filter.xsd ogc/filter/1.1.0/filter.xsd
--- ../resources-original/ogc/filter/1.1.0/filter.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/filter/1.1.0/filter.xsd	Sun Jul 10 18:00:58 2016
@@ -198,9 +198,8 @@
       <xsd:complexContent>
          <xsd:extension base="ogc:SpatialOpsType">
             <xsd:sequence>
-               <xsd:element ref="ogc:PropertyName"/>
-               <xsd:choice>
-                  <xsd:element ref="ogc:PropertyName"/>
+               <xsd:element ref="ogc:PropertyName" maxOccurs="2"/>
+               <xsd:choice minOccurs="0">
                   <xsd:element ref="gml:_Geometry"/>
                   <xsd:element ref="gml:Envelope"/>
                </xsd:choice>
diff -urN ../resources-original/ogc/gml/2.1.2/feature.xsd ogc/gml/2.1.2/feature.xsd
--- ../resources-original/ogc/gml/2.1.2/feature.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/gml/2.1.2/feature.xsd	Sun Jul 10 18:00:58 2016
@@ -24,6 +24,7 @@
   <element name="_Feature" type="gml:AbstractFeatureType" abstract="true"/>
   <element name="_FeatureCollection" type="gml:AbstractFeatureCollectionType" abstract="true" substitutionGroup="gml:_Feature"/>
   <element name="featureMember" type="gml:FeatureAssociationType"/>
+  <element name="otherFeatureMember" type="gml:FeatureAssociationType" substitutionGroup="gml:featureMember"/>
   <!-- some basic geometric properties of features -->
   <element name="_geometryProperty" type="gml:GeometryAssociationType" abstract="true"/>
   <element name="geometryProperty" type="gml:GeometryAssociationType"/>
diff -urN ../resources-original/ogc/gml/3.1.1/base/feature.xsd ogc/gml/3.1.1/base/feature.xsd
--- ../resources-original/ogc/gml/3.1.1/base/feature.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/gml/3.1.1/base/feature.xsd	Sun Jul 10 18:00:58 2016
@@ -71,6 +71,7 @@
 	<!-- ====================================================================== -->
 	<!-- ===== property for feature association ==== -->
 	<element name="featureMember" type="gml:FeaturePropertyType"/>
+	<element name="otherFeatureMember" type="gml:FeaturePropertyType" substitutionGroup="gml:featureMember"/>
 	<element name="featureProperty" type="gml:FeaturePropertyType"/>
 	<!-- ============================================================== -->
 	<complexType name="FeaturePropertyType">
@@ -85,6 +86,7 @@
 	<!-- ============================================================== -->
 	<!-- ===== property for association of an array of features ===== -->
 	<element name="featureMembers" type="gml:FeatureArrayPropertyType"/>
+	<element name="otherFeatureMembers" type="gml:FeatureArrayPropertyType" substitutionGroup="gml:featureMembers"/>
 	<!-- =========================================================== -->
 	<complexType name="FeatureArrayPropertyType">
 		<annotation>
diff -urN ../resources-original/ogc/gml/3.2.1/deprecatedTypes.xsd ogc/gml/3.2.1/deprecatedTypes.xsd
--- ../resources-original/ogc/gml/3.2.1/deprecatedTypes.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/gml/3.2.1/deprecatedTypes.xsd	Sun Jul 10 18:00:58 2016
@@ -807,6 +807,7 @@
 			<appinfo>deprecated</appinfo>
 		</annotation>
 	</element>
+	<element name="otherFeatureMember" type="gml:FeaturePropertyType" substitutionGroup="gml:featureMember"/>
 	<element name="featureProperty" type="gml:FeaturePropertyType">
 		<annotation>
 			<appinfo>deprecated</appinfo>
@@ -825,6 +826,7 @@
 			<appinfo>deprecated</appinfo>
 		</annotation>
 	</element>
+	<element name="otherFeatureMembers" type="gml:FeatureArrayPropertyType" substitutionGroup="gml:featureMembers"/>
 	<complexType name="BoundedFeatureType" abstract="true">
 		<annotation>
 			<appinfo>deprecated</appinfo>
diff -urN ../resources-original/ogc/kml/2.3/ogckml23.xsd ogc/kml/2.3/ogckml23.xsd
--- ../resources-original/ogc/kml/2.3/ogckml23.xsd	Thu Oct 22 16:14:09 2015
+++ ogc/kml/2.3/ogckml23.xsd	Sun Jul 10 18:00:58 2016
@@ -23,9 +23,9 @@
   <import namespace="urn:oasis:names:tc:ciq:xsdschema:xAL:2.0"
     schemaLocation="http://docs.oasis-open.org/election/external/xAL.xsd"/>
 
-  <defaultOpenContent mode="interleave">
+  <!--defaultOpenContent mode="interleave">
     <any namespace="##other" processContents="lax"/>
-  </defaultOpenContent>
+  </defaultOpenContent-->
 
   <!-- KML field types (simple content) -->
 
@@ -767,167 +767,167 @@
     <attribute name="version" type="kml:kmlVersionType" default="2.2"/>
     <anyAttribute namespace="##other" processContents="lax"/>
     <!-- TODO(sean): Make sure these are all the parent elements we care about. -->
-    <assert test="if (.//kml:Track)
+    <!-- assert test="if (.//kml:Track)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:Track is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:MultiTrack)
+    </assert-->
+    <!-- assert test="if (.//kml:MultiTrack)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:MultiTrack is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:Tour)
+    </assert-->
+    <!-- assert test="if (.//kml:Tour)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:Tour is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:LatLonQuad)
+    </assert-->
+    <!-- assert test="if (.//kml:LatLonQuad)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:LatLonQuad is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:seaFloorAltitudeMode)
+    </assert-->
+    <!-- assert test="if (.//kml:seaFloorAltitudeMode)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:seaFloorAltitudeMode is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:altitudeOffset)
+    </assert-->
+    <!-- assert test="if (.//kml:altitudeOffset)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:altitudeOffset is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:balloonVisibility)
+    </assert-->
+    <!-- assert test="if (.//kml:balloonVisibility)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:balloonVisibility is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:Camera/kml:TimeStamp)
+    </assert-->
+    <!-- assert test="if (.//kml:Camera/kml:TimeStamp)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:TimeStamp is not a valid child of kml:Camera in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:Camera/kml:TimeSpan)
+    </assert-->
+    <!-- assert test="if (.//kml:Camera/kml:TimeSpan)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:TimeSpan is not a valid child of kml:Camera in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:LookAt/kml:TimeStamp)
+    </assert-->
+    <!-- assert test="if (.//kml:LookAt/kml:TimeStamp)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:TimeStamp is not a valid child of kml:LookAt in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:LookAt/kml:TimeSpan)
+    </assert-->
+    <!-- assert test="if (.//kml:LookAt/kml:TimeSpan)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:LookAt is not a valid child of kml:LookAt in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-   <assert test="if (.//kml:SimpleArrayField)
+    </assert-->
+   <!-- assert test="if (.//kml:SimpleArrayField)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:SimpleArrayField is not allowed in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:SimpleField/@uom)
+   </assert-->   
+   <!-- assert test="if (.//kml:SimpleField/@uom)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>Attribute 'uom' is not a valid child of kml:SimpleField in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Data/@uom)
+   </assert-->   
+   <!-- assert test="if (.//kml:Data/@uom)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>Attribute 'uom' is not a valid child of kml:Data in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Camera/kml:horizFov)
+   </assert-->   
+   <!-- assert test="if (.//kml:Camera/kml:horizFov)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:horizFov is not a valid child of kml:Camera in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:LookAt/kml:horizFov)
+   </assert-->   
+   <!-- assert test="if (.//kml:LookAt/kml:horizFov)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:horizFov is not a valid child of kml:LookAt in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Create/kml:MultiGeometry)
+   </assert-->   
+   <!-- assert test="if (.//kml:Create/kml:MultiGeometry)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:MultiGeometry is not a valid child of kml:Create in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:MultiGeometry)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:MultiGeometry)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:MultiGeometry is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:Point)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:Point)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:Point is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:LineString)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:LineString)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:LineString is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:LinearRing)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:LinearRing)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:LinearRing is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:Polygon)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:Polygon)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:Polygon is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:Model)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:Model)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:Model is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
+   </assert-->   
   </complexType>
   <element name="KmlSimpleExtensionGroup" abstract="true" type="anySimpleType"/>
   <element name="KmlObjectExtensionGroup" abstract="true"
@@ -1105,16 +1105,16 @@
           <element ref="kml:RegionObjectExtensionGroup" minOccurs="0"
             maxOccurs="unbounded"/>
         </all>
-        <assert test="not(./kml:LatLonBox) ">
+        <!-- assert test="not(./kml:LatLonBox) ">
           <annotation>
             <documentation>kml:LatLonBox is prohibited in kml:Region. Use kml:LatLonAltBox instead.</documentation>
           </annotation>
-        </assert>
-        <assert test="not(./kml:LatLonQuad) ">
+        </assert-->
+        <!-- assert test="not(./kml:LatLonQuad) ">
           <annotation>
             <documentation>kml:LatLonQuad is prohibited in kml:Region. Use kml:LatLonAltBox instead.</documentation>
           </annotation>
-        </assert>
+        </assert-->
       </extension>
     </complexContent>
   </complexType>
@@ -1378,16 +1378,16 @@
           <element ref="kml:TrackObjectExtensionGroup" minOccurs="0"
             maxOccurs="unbounded"/>
         </all>
-        <assert test="count(./kml:when) = count(./kml:coord)">
+        <!-- assert test="count(./kml:when) = count(./kml:coord)">
           <annotation>
             <documentation>kml:Track must have an equal number of kml:when and kml:coord elements.</documentation>
           </annotation>
-        </assert>
-        <assert test="if(./kml:angles) then count(./kml:angles) = count(./kml:when) else true()">
+        </assert-->
+        <!-- assert test="if(./kml:angles) then count(./kml:angles) = count(./kml:when) else true()">
           <annotation>
             <documentation>If kml:angles appears in kml:Track there must be an equal number of kml:angles and kml:when elements.</documentation>
           </annotation>
-        </assert>
+        </assert-->
       </extension>
     </complexContent>
   </complexType>
@@ -1540,11 +1540,11 @@
           <element ref="kml:GroundOverlayObjectExtensionGroup" minOccurs="0"
             maxOccurs="unbounded"/>
         </all>
-        <assert test="not(./kml:LatLonAltBox)">
+        <!-- assert test="not(./kml:LatLonAltBox)">
           <annotation>
             <documentation>kml:LatLonAltBox is prohibited in kml:GroundOverlay. Use kml:LatLonBox or kml:LatLonQuad instead.</documentation>
           </annotation>
-        </assert>
+        </assert-->
       </extension>
     </complexContent>
   </complexType>
diff -urN ../resources-original/ogc/ols/1.1.0/ADT.xsd ogc/ols/1.1.0/ADT.xsd
--- ../resources-original/ogc/ols/1.1.0/ADT.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/ADT.xsd	Sun Jul 10 18:00:58 2016
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="geometry.xsd"/>
 	<!--  The ADTs defined in this document represent the minimum set of essential elements as determined from the analysis of the APIs provided by OpenLS sponsors and participants. There are many additional elements that may be added but these are considered to be optional. The APIs considered include: Oracle, Webraska, ESRI, Vodafone, opt[e]way, NavTech, IntelliWhere, and LocatioNet. -->
 	<!--=============================================================================
diff -urN ../resources-original/ogc/ols/1.1.0/DirectoryService.xsd ogc/ols/1.1.0/DirectoryService.xsd
--- ../resources-original/ogc/ols/1.1.0/DirectoryService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/DirectoryService.xsd	Sun Jul 10 18:00:58 2016
@@ -5,7 +5,8 @@
  Copyright (c) 2006,2010 Open Geospatial Consortium.
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="DirectoryRequest" type="xls:DirectoryRequestType" substitutionGroup="xls:_RequestParameters"/>
diff -urN ../resources-original/ogc/ols/1.1.0/GatewayService.xsd ogc/ols/1.1.0/GatewayService.xsd
--- ../resources-original/ogc/ols/1.1.0/GatewayService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/GatewayService.xsd	Sun Jul 10 18:00:58 2016
@@ -5,7 +5,8 @@
  Copyright (c) 2006,2010 Open Geospatial Consortium.
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!--================= Gateway Request and Response Context=========================-->
 	<element name="_GatewayParameters" type="xls:AbstractGatewayParametersType" abstract="false">
diff -urN ../resources-original/ogc/ols/1.1.0/LocationUtilityService.xsd ogc/ols/1.1.0/LocationUtilityService.xsd
--- ../resources-original/ogc/ols/1.1.0/LocationUtilityService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/LocationUtilityService.xsd	Sun Jul 10 18:00:58 2016
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="GeocodeRequest" type="xls:GeocodeRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.1.0/PresentationService.xsd ogc/ols/1.1.0/PresentationService.xsd
--- ../resources-original/ogc/ols/1.1.0/PresentationService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/PresentationService.xsd	Sun Jul 10 18:00:58 2016
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ==============================================GetCapabilities Request=====================-->
 	<element name="GetPortrayMapCapabilitiesRequest" type="xls:GetPortrayMapCapabilitiesRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.1.0/RouteService.xsd ogc/ols/1.1.0/RouteService.xsd
--- ../resources-original/ogc/ols/1.1.0/RouteService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/RouteService.xsd	Sun Jul 10 18:00:58 2016
@@ -7,7 +7,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ======================================================================================
 				Route Service Basic
diff -urN ../resources-original/ogc/ols/1.1.0/geometry.xsd ogc/ols/1.1.0/geometry.xsd
--- ../resources-original/ogc/ols/1.1.0/geometry.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.1.0/geometry.xsd	Sun Jul 10 18:00:58 2016
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<include schemaLocation="UOM.xsd"/>
 	<!-- =============================== type ===================================== -->
diff -urN ../resources-original/ogc/ols/1.2/ADT.xsd ogc/ols/1.2/ADT.xsd
--- ../resources-original/ogc/ols/1.2/ADT.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/ADT.xsd	Sun Jul 10 18:00:58 2016
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="geometry.xsd"/>
 	<!--  The ADTs defined in this document represent the minimum set of essential elements as determined from the analysis of the APIs provided by OpenLS sponsors and participants. There are many additional elements that may be added but these are considered to be optional. The APIs considered include: Oracle, Webraska, ESRI, Vodafone, opt[e]way, NavTech, IntelliWhere, and LocatioNet. -->
 	<!--=============================================================================
diff -urN ../resources-original/ogc/ols/1.2/DirectoryService.xsd ogc/ols/1.2/DirectoryService.xsd
--- ../resources-original/ogc/ols/1.2/DirectoryService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/DirectoryService.xsd	Sun Jul 10 18:00:58 2016
@@ -9,7 +9,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="DirectoryRequest" type="xls:DirectoryRequestType" substitutionGroup="xls:_RequestParameters"/>
diff -urN ../resources-original/ogc/ols/1.2/GatewayService.xsd ogc/ols/1.2/GatewayService.xsd
--- ../resources-original/ogc/ols/1.2/GatewayService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/GatewayService.xsd	Sun Jul 10 18:00:58 2016
@@ -9,7 +9,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!--================= Gateway Request and Response Context=========================-->
 	<element name="_GatewayParameters" type="xls:AbstractGatewayParametersType" abstract="false">
diff -urN ../resources-original/ogc/ols/1.2/LocationUtilityService.xsd ogc/ols/1.2/LocationUtilityService.xsd
--- ../resources-original/ogc/ols/1.2/LocationUtilityService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/LocationUtilityService.xsd	Sun Jul 10 18:00:58 2016
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="GeocodeRequest" type="xls:GeocodeRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.2/PresentationService.xsd ogc/ols/1.2/PresentationService.xsd
--- ../resources-original/ogc/ols/1.2/PresentationService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/PresentationService.xsd	Sun Jul 10 18:00:58 2016
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ==============================================GetCapabilities Request=====================-->
 	<element name="GetPortrayMapCapabilitiesRequest" type="xls:GetPortrayMapCapabilitiesRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.2/RouteService.xsd ogc/ols/1.2/RouteService.xsd
--- ../resources-original/ogc/ols/1.2/RouteService.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/RouteService.xsd	Sun Jul 10 18:00:58 2016
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ======================================================================================
 				Route Service Basic
diff -urN ../resources-original/ogc/ols/1.2/geometry.xsd ogc/ols/1.2/geometry.xsd
--- ../resources-original/ogc/ols/1.2/geometry.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ols/1.2/geometry.xsd	Sun Jul 10 18:00:58 2016
@@ -11,7 +11,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/> 
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/> 
 	<include schemaLocation="UOM.xsd"/> 
 	<!-- =============================== type ===================================== --> 
diff -urN ../resources-original/ogc/ols/1.3/nav-Positioning.xsd ogc/ols/1.3/nav-Positioning.xsd
--- ../resources-original/ogc/ols/1.3/nav-Positioning.xsd	Thu Jul 31 20:46:15 2014
+++ ogc/ols/1.3/nav-Positioning.xsd	Sun Jul 10 18:00:58 2016
@@ -8,7 +8,8 @@
 		</xsd:documentation>
 	</xsd:annotation>
 	<xsd:include schemaLocation="nav-All.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/>
+	<!--xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/-->
+	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<xsd:import namespace="http://www.opengis.net/xls" schemaLocation="http://schemas.opengis.net/ols/1.2/XLS.xsd"/>
 	<xsd:include schemaLocation="nav-CursorUtil.xsd"/>
 	<xsd:include schemaLocation="nav-UOM.xsd"/>
diff -urN ../resources-original/ogc/ols/1.3/nav-RouteService.xsd ogc/ols/1.3/nav-RouteService.xsd
--- ../resources-original/ogc/ols/1.3/nav-RouteService.xsd	Thu Jul 31 20:46:15 2014
+++ ogc/ols/1.3/nav-RouteService.xsd	Sun Jul 10 18:00:58 2016
@@ -8,7 +8,8 @@
 		</documentation>
 	</annotation>
 	<include schemaLocation="nav-All.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<import namespace="http://www.opengis.net/xls" schemaLocation="http://schemas.opengis.net/ols/1.2/XLS.xsd"/>
 	<import namespace="http://www.opengis.net/xls" schemaLocation="http://schemas.opengis.net/ols/1.2/RouteService.xsd"/>
 	<include schemaLocation="nav-Positioning.xsd"/>
diff -urN ../resources-original/ogc/om/2.0/observation.xsd ogc/om/2.0/observation.xsd
--- ../resources-original/ogc/om/2.0/observation.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/om/2.0/observation.xsd	Sun Jul 10 18:00:58 2016
@@ -429,4 +429,14 @@
 	<!-- ====================================================================== -->
 	<!-- ====================================================================== -->
 	<!-- ====================================================================== -->
+
+	<element name="OtherAbstractTimeObject" type="om:OtherAbstractTimeObjectType" substitutionGroup="gml:AbstractTimeObject"/>
+	<complexType
+		name="OtherAbstractTimeObjectType">
+		<complexContent>
+			<extension
+				base="gml:AbstractTimeObjectType"/>
+		</complexContent>
+	</complexType>
+
 </schema>
diff -urN ../resources-original/ogc/ows/1.1.0/owsContents.xsd ogc/ows/1.1.0/owsContents.xsd
--- ../resources-original/ogc/ows/1.1.0/owsContents.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ows/1.1.0/owsContents.xsd	Sun Jul 10 18:00:58 2016
@@ -84,4 +84,8 @@
 		</complexContent>
 	</complexType>
 	<!-- ===========================================================-->
+	
+	<!-- This is to force JAXB generate reference properties -->
+	<element name="DatasetDescriptionSummaryExtension" type="ows:DatasetDescriptionSummaryBaseType" substitutionGroup="ows:DatasetDescriptionSummary"/>
+	<element name="OtherSourceExtension" type="ows:MetadataType" substitutionGroup="ows:OtherSource"/>
 </schema>
diff -urN ../resources-original/ogc/ows/2.0/owsAdditionalParameters.xsd ogc/ows/2.0/owsAdditionalParameters.xsd
--- ../resources-original/ogc/ows/2.0/owsAdditionalParameters.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/ows/2.0/owsAdditionalParameters.xsd	Sun Jul 10 18:00:58 2016
@@ -53,13 +53,13 @@
   <!-- ========================================================== -->
   <complexType name="AdditionalParametersType">
     <complexContent>
-      <extension base="ows:AdditionalParametersBaseType">
+      <restriction base="ows:AdditionalParametersBaseType">
         <sequence>
           <element ref="ows:AdditionalParameter"
                    minOccurs="0"
                    maxOccurs="unbounded" />
         </sequence>
-      </extension>
+      </restriction>
     </complexContent>
   </complexType>
   <!-- ========================================================== -->
diff -urN ../resources-original/ogc/sweCommon/1.0.0/aggregateTypes.xsd ogc/sweCommon/1.0.0/aggregateTypes.xsd
--- ../resources-original/ogc/sweCommon/1.0.0/aggregateTypes.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/sweCommon/1.0.0/aggregateTypes.xsd	Sun Jul 10 18:00:58 2016
@@ -139,7 +139,7 @@
 		<xs:annotation>
 			<xs:documentation>Use to point or include data values inline</xs:documentation>
 		</xs:annotation>
-		<xs:complexContent>
+		<xs:complexContent mixed="true">
 			<xs:extension base="xs:anyType">
 				<xs:attributeGroup ref="gml:AssociationAttributeGroup"/>
 			</xs:extension>
diff -urN ../resources-original/ogc/sweCommon/1.0.1/aggregateTypes.xsd ogc/sweCommon/1.0.1/aggregateTypes.xsd
--- ../resources-original/ogc/sweCommon/1.0.1/aggregateTypes.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/sweCommon/1.0.1/aggregateTypes.xsd	Sun Jul 10 18:00:58 2016
@@ -138,7 +138,7 @@
 		<xs:annotation>
 			<xs:documentation>Use to point or include data values inline</xs:documentation>
 		</xs:annotation>
-		<xs:complexContent>
+		<xs:complexContent mixed="true">
 			<xs:extension base="xs:anyType">
 				<xs:attribute name="recordCount" type="xs:positiveInteger"/>
 				<xs:attributeGroup ref="gml:AssociationAttributeGroup"/>
diff -urN ../resources-original/ogc/sweCommon/2.0/basic_types.xsd ogc/sweCommon/2.0/basic_types.xsd
--- ../resources-original/ogc/sweCommon/2.0/basic_types.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/sweCommon/2.0/basic_types.xsd	Sun Jul 10 18:00:58 2016
@@ -78,7 +78,7 @@
 	</complexType>
 	<!-- ================================================= -->
 	<complexType name="EncodedValuesPropertyType">
-		<complexContent>
+		<complexContent mixed="true">
 			<extension base="anyType">
 			  <attributeGroup ref="swe:AssociationAttributeGroup"/>
 			</extension>
diff -urN ../resources-original/ogc/swes/2.0/swesContents.xsd ogc/swes/2.0/swesContents.xsd
--- ../resources-original/ogc/swes/2.0/swesContents.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/swes/2.0/swesContents.xsd	Sun Jul 10 18:00:58 2016
@@ -16,6 +16,8 @@
       <documentation>contains metadata about a procedure / sensor hosted by the service</documentation>
     </annotation>
   </element>
+  <!-- Other AbstractOffering element to force XJC generate element reference property -->
+  <element name="OtherOffering" type="swes:AbstractOfferingType" abstract="true" substitutionGroup="swes:AbstractOffering"/>
   <complexType name="AbstractOfferingType" abstract="true">
     <complexContent>
       <extension base="swes:AbstractSWESType">
diff -urN ../resources-original/ogc/tml/1.0.0/tml.xsd ogc/tml/1.0.0/tml.xsd
--- ../resources-original/ogc/tml/1.0.0/tml.xsd	Fri Aug 14 22:02:25 2009
+++ ogc/tml/1.0.0/tml.xsd	Sun Jul 10 18:00:58 2016
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- edited with XMLSpy v2006 U (http://www.altova.com) by Steve Havens (IRIS Corporation) -->
 <xs:schema xmlns:tml="http://www.opengis.net/tml" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ism="urn:us:gov:ic:ism:v2" targetNamespace="http://www.opengis.net/tml" elementFormDefault="qualified" attributeFormDefault="unqualified">
-	<xs:import namespace="urn:us:gov:ic:ism:v2" schemaLocation="IC-ISM-v2.xsd"/>
+	<xs:import namespace="urn:us:gov:ic:ism:v2" schemaLocation="http://schemas.opengis.net/ic/2.0/IC-ISM-v2.xsd"/>
 	<xs:element name="tml">
 		<xs:annotation>
 			<xs:documentation>Root Element.  Also contains attributes to indicate the overall security classification of this TML stream or file.  If needed individual data clusters can be labeled with a security class.</xs:documentation>
diff -urN ../resources-original/ogc/wcs/1.0.0/wcsCapabilities.xsd ogc/wcs/1.0.0/wcsCapabilities.xsd
--- ../resources-original/ogc/wcs/1.0.0/wcsCapabilities.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/wcs/1.0.0/wcsCapabilities.xsd	Sun Jul 10 18:00:58 2016
@@ -444,9 +444,9 @@
         </xs:annotation>
         <xs:complexContent>
             <xs:restriction base="gml:AbstractGMLType">
-                <xs:sequence>
+                <!--xs:sequence>
                     <xs:element ref="metadataLink" minOccurs="0" maxOccurs="unbounded"/>
-                </xs:sequence>
+                </xs:sequence-->
             </xs:restriction>
         </xs:complexContent>
     </xs:complexType>
diff -urN ../resources-original/ogc/wcs/interpolation/1.0/wcsInt.xsd ogc/wcs/interpolation/1.0/wcsInt.xsd
--- ../resources-original/ogc/wcs/interpolation/1.0/wcsInt.xsd	Tue Nov 18 21:32:21 2014
+++ ogc/wcs/interpolation/1.0/wcsInt.xsd	Sun Jul 10 18:00:58 2016
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema targetNamespace="http://www.opengis.net/WCS_service-extension_interpolation/1.0" 
-    xmlns:int="http://www.opengis.net/WCS_service-extension_interpolation/1.0"
+<schema targetNamespace="http://www.opengis.net/wcs/interpolation/1.0" 
+    xmlns:int="http://www.opengis.net/wcs/interpolation/1.0"
     xmlns="http://www.w3.org/2001/XMLSchema" 
     elementFormDefault="qualified" version="1.0.0" xml:lang="en">
     <annotation>
diff -urN ../resources-original/ogc/wcs/processing/2.0/wcsProcessCoverages.xsd ogc/wcs/processing/2.0/wcsProcessCoverages.xsd
--- ../resources-original/ogc/wcs/processing/2.0/wcsProcessCoverages.xsd	Wed Nov 26 16:41:48 2014
+++ ogc/wcs/processing/2.0/wcsProcessCoverages.xsd	Sun Jul 10 18:00:59 2016
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema targetNamespace="http://www.opengis.net/wcs_service-extension_processing/2.0"
-    xmlns:proc="http://www.opengis.net/wcs_service-extension_processing/2.0"
+<xs:schema targetNamespace="http://www.opengis.net/wcs/processing/2.0"
+    xmlns:proc="http://www.opengis.net/wcs/processing/2.0"
     xmlns:wcs="http://www.opengis.net/wcs/2.0"
     xmlns:gmlcov="http://www.opengis.net/gmlcov/1.0"
     xmlns:gml="http://www.opengis.net/gml/3.2.1"
diff -urN ../resources-original/ogc/wcs/scaling/1.0/examples/scaleAxesByFactor.xml ogc/wcs/scaling/1.0/examples/scaleAxesByFactor.xml
--- ../resources-original/ogc/wcs/scaling/1.0/examples/scaleAxesByFactor.xml	Wed Nov 19 21:22:57 2014
+++ ogc/wcs/scaling/1.0/examples/scaleAxesByFactor.xml	Sun Jul 10 18:00:59 2016
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/WCS_service-extension_scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/WCS_service-extension_scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
+<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/wcs/scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/wcs/scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
 	<wcs:Extension>
 		<scal:Scaling>
 			<scal:ScaleAxesByFactor>
diff -urN ../resources-original/ogc/wcs/scaling/1.0/examples/scaleByFactor.xml ogc/wcs/scaling/1.0/examples/scaleByFactor.xml
--- ../resources-original/ogc/wcs/scaling/1.0/examples/scaleByFactor.xml	Wed Nov 19 21:22:57 2014
+++ ogc/wcs/scaling/1.0/examples/scaleByFactor.xml	Sun Jul 10 18:00:59 2016
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/WCS_service-extension_scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/WCS_service-extension_scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
+<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/wcs/scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/wcs/scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
 	<wcs:Extension>
 		<scal:Scaling>
 			<scal:ScaleByFactor>
diff -urN ../resources-original/ogc/wcs/scaling/1.0/examples/scaleToExtent.xml ogc/wcs/scaling/1.0/examples/scaleToExtent.xml
--- ../resources-original/ogc/wcs/scaling/1.0/examples/scaleToExtent.xml	Wed Nov 19 21:22:57 2014
+++ ogc/wcs/scaling/1.0/examples/scaleToExtent.xml	Sun Jul 10 18:00:59 2016
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/WCS_service-extension_scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/WCS_service-extension_scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
+<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/wcs/scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/wcs/scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
 	<wcs:Extension>
 		<scal:Scaling>
 			<scal:ScaleToExtent>
diff -urN ../resources-original/ogc/wcs/scaling/1.0/examples/scaleToSize.xml ogc/wcs/scaling/1.0/examples/scaleToSize.xml
--- ../resources-original/ogc/wcs/scaling/1.0/examples/scaleToSize.xml	Wed Nov 19 21:22:57 2014
+++ ogc/wcs/scaling/1.0/examples/scaleToSize.xml	Sun Jul 10 18:00:59 2016
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/WCS_service-extension_scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/WCS_service-extension_scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
+<wcs:GetCoverage xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:scal="http://www.opengis.net/wcs/scaling/1.0" service="WCS" version="2.0.1" xsi:schemaLocation="http://www.opengis.net/wcs/scaling/1.0 http://schemas.opengis.net/wcs/scaling/1.0/wcsScal.xsd">
 	<wcs:Extension>
 		<scal:Scaling>
 			<scal:ScaleToSize>
diff -urN ../resources-original/ogc/wcs/scaling/1.0/wcsScal.xsd ogc/wcs/scaling/1.0/wcsScal.xsd
--- ../resources-original/ogc/wcs/scaling/1.0/wcsScal.xsd	Tue Nov 18 17:14:18 2014
+++ ogc/wcs/scaling/1.0/wcsScal.xsd	Sun Jul 10 18:00:59 2016
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema xmlns:scal="http://www.opengis.net/WCS_service-extension_scaling/1.0" xmlns="http://www.w3.org/2001/XMLSchema" xmlns:ns1="http://www.opengis.net/wcs/2.0" targetNamespace="http://www.opengis.net/WCS_service-extension_scaling/1.0" elementFormDefault="qualified" version="1.0.0" xml:lang="en">
+<schema xmlns:scal="http://www.opengis.net/wcs/scaling/1.0" xmlns="http://www.w3.org/2001/XMLSchema" xmlns:wcs="http://www.opengis.net/wcs/2.0" targetNamespace="http://www.opengis.net/wcs/scaling/1.0" elementFormDefault="qualified" version="1.0.0" xml:lang="en">
 	<annotation>
 		<appinfo>scal.xsd</appinfo>
 		<documentation>This XML Schema Document is part of the WCS Scaling Extension [OGC 12-039]. It encodes the elements and types that allow expressing Scaling in WCS client/server communication.
diff -urN ../resources-original/ogc/wfs/1.0.0/WFS-capabilities.xsd ogc/wfs/1.0.0/WFS-capabilities.xsd
--- ../resources-original/ogc/wfs/1.0.0/WFS-capabilities.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/wfs/1.0.0/WFS-capabilities.xsd	Sun Jul 10 19:28:31 2016
@@ -54,12 +54,13 @@
    <!-- A human-readable title to briefly identify
         this server in menus. -->
    <xsd:element name="Title" type="xsd:string"/>
-
+   <!--
    <xsd:element name="Query"  type="wfs:EmptyType"/>
    <xsd:element name="Insert" type="wfs:EmptyType"/>
    <xsd:element name="Update" type="wfs:EmptyType"/>
    <xsd:element name="Delete" type="wfs:EmptyType"/>
    <xsd:element name="Lock"   type="wfs:EmptyType"/>
+   -->
 
    <!-- REDEFINE THIS ELEMENT AS NEEDED IN YOUR XML  -->
    <xsd:element name="VendorSpecificCapabilities" type="xsd:string"/>
@@ -155,26 +156,26 @@
    <xsd:complexType name="RequestType">
       <xsd:choice maxOccurs="unbounded">
          <xsd:element name="GetCapabilities"
-                      type="wfs:GetCapabilitiesType"/>
+                      type="wfs:Capabilities_GetCapabilitiesType"/>
          <xsd:element name="DescribeFeatureType"
-                      type="wfs:DescribeFeatureTypeType"/>
+                      type="wfs:Capabilities_DescribeFeatureTypeType"/>
          <xsd:element name="Transaction"
-                      type="wfs:TransactionType"/>
+                      type="wfs:Capabilities_TransactionType"/>
          <xsd:element name="GetFeature"
-                      type="wfs:GetFeatureTypeType"/>
+                      type="wfs:Capabilities_GetFeatureTypeType"/>
          <xsd:element name="GetFeatureWithLock"
-                      type="wfs:GetFeatureTypeType"/>
+                      type="wfs:Capabilities_GetFeatureTypeType"/>
          <xsd:element name="LockFeature"
-                      type="wfs:LockFeatureTypeType"/>
+                      type="wfs:Capabilities_LockFeatureTypeType"/>
       </xsd:choice>
    </xsd:complexType>
-   <xsd:complexType name="GetCapabilitiesType">
+   <xsd:complexType name="Capabilities_GetCapabilitiesType">
       <xsd:sequence>
          <xsd:element name="DCPType"
                       type="wfs:DCPTypeType" maxOccurs="unbounded"/>
       </xsd:sequence>
    </xsd:complexType>
-   <xsd:complexType name="DescribeFeatureTypeType">
+   <xsd:complexType name="Capabilities_DescribeFeatureTypeType">
       <xsd:sequence>
          <xsd:element name="SchemaDescriptionLanguage"
                       type="wfs:SchemaDescriptionLanguageType"/>
@@ -182,13 +183,13 @@
                       type="wfs:DCPTypeType" maxOccurs="unbounded"/>
       </xsd:sequence>
    </xsd:complexType>
-   <xsd:complexType name="TransactionType">
+   <xsd:complexType name="Capabilities_TransactionType">
       <xsd:sequence>
          <xsd:element name="DCPType"
                       type="wfs:DCPTypeType" maxOccurs="unbounded"/>
       </xsd:sequence>
    </xsd:complexType>
-   <xsd:complexType name="GetFeatureTypeType">
+   <xsd:complexType name="Capabilities_GetFeatureTypeType">
       <xsd:sequence>
          <xsd:element name="ResultFormat"
                       type="wfs:ResultFormatType"/>
@@ -196,7 +197,7 @@
                       type="wfs:DCPTypeType" maxOccurs="unbounded"/>
       </xsd:sequence>
    </xsd:complexType>
-   <xsd:complexType name="LockFeatureTypeType">
+   <xsd:complexType name="Capabilities_LockFeatureTypeType">
       <xsd:sequence>
          <xsd:element name="DCPType"
                       type="wfs:DCPTypeType" maxOccurs="unbounded"/>
@@ -322,11 +323,11 @@
    </xsd:complexType>
    <xsd:complexType name="OperationsType">
       <xsd:choice maxOccurs="unbounded">
-         <xsd:element ref="wfs:Insert"/>
-         <xsd:element ref="wfs:Update"/>
-         <xsd:element ref="wfs:Delete"/>
-         <xsd:element ref="wfs:Query"/>
-         <xsd:element ref="wfs:Lock"/>
+         <xsd:element name="Insert" type="wfs:EmptyType"/>
+         <xsd:element name="Update" type="wfs:EmptyType"/>
+         <xsd:element name="Delete" type="wfs:EmptyType"/>
+         <xsd:element name="Query" type="wfs:EmptyType"/>
+         <xsd:element name="Lock" type="wfs:EmptyType"/>
       </xsd:choice>
    </xsd:complexType>
    <xsd:complexType name="PostType">
@@ -355,5 +356,7 @@
          <xsd:element ref="wfs:XMLSCHEMA"/>
       </xsd:sequence>
    </xsd:complexType>
+   <!--
    <xsd:complexType name="EmptyType" />
+   -->
 </xsd:schema>
diff -urN ../resources-original/ogc/wms/1.3.0/capabilities_1_3_0.xsd ogc/wms/1.3.0/capabilities_1_3_0.xsd
--- ../resources-original/ogc/wms/1.3.0/capabilities_1_3_0.xsd	Sun Jul 22 03:59:38 2012
+++ ogc/wms/1.3.0/capabilities_1_3_0.xsd	Sun Jul 10 18:00:59 2016
@@ -615,4 +615,7 @@
 			<maxInclusive value="90"/>
 		</restriction>
 	</simpleType>
+
+	<element name="OtherExtendedCapabilities" substitutionGroup="wms:_ExtendedCapabilities"/>
+	<element name="OtherExtendedOperation" type="wms:OperationType" substitutionGroup="wms:_ExtendedOperation"/>
 </schema>
