diff --git ogc/csw/2.0.2/CSW-discovery.xsd ogc/csw/2.0.2/CSW-discovery.xsd
index ab0f10b..5c8164b 100644
--- ogc/csw/2.0.2/CSW-discovery.xsd
+++ ogc/csw/2.0.2/CSW-discovery.xsd
@@ -25,9 +25,9 @@
    <xsd:include schemaLocation="csw.xsd"/>
    <xsd:include schemaLocation="record.xsd"/>
 
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.1.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.1.0/filter.xsd"/>
 
    <xsd:complexType name="RequestBaseType" id="RequestBaseType" abstract="true">
       <xsd:annotation>
diff --git ogc/csw/2.0.2/rec-dcterms.xsd ogc/csw/2.0.2/rec-dcterms.xsd
index f2450e3..d4f1ae6 100644
--- ogc/csw/2.0.2/rec-dcterms.xsd
+++ ogc/csw/2.0.2/rec-dcterms.xsd
@@ -9,7 +9,7 @@
          source="http://dublincore.org/documents/dcmi-terms/">This schema declares additional DCMI elements and element refinements 
     in the "http://purl.org/dc/terms/" namespace.</xs:documentation>
    </xs:annotation>
-   <xs:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="http://schemas.opengis.net/csw/2.0.2/rec-dcmes.xsd"/>
+   <xs:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="../../csw/2.0.2/rec-dcmes.xsd"/>
    <xs:element name="abstract" type="dc:SimpleLiteral"
       substitutionGroup="dc:description"/>
    <xs:element name="accessRights" type="dc:SimpleLiteral"
diff --git ogc/csw/2.0.2/record.xsd ogc/csw/2.0.2/record.xsd
index 6919b02..0be90d6 100644
--- ogc/csw/2.0.2/record.xsd
+++ ogc/csw/2.0.2/record.xsd
@@ -22,9 +22,9 @@
    </xsd:annotation>
 
    <xsd:include schemaLocation="csw.xsd"/>
-   <xsd:import namespace="http://purl.org/dc/terms/" schemaLocation="http://schemas.opengis.net/csw/2.0.2/rec-dcterms.xsd"/>
-   <xsd:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="http://schemas.opengis.net/csw/2.0.2/rec-dcmes.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://purl.org/dc/terms/" schemaLocation="../../csw/2.0.2/rec-dcterms.xsd"/>
+   <xsd:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="../../csw/2.0.2/rec-dcmes.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
    <xsd:element name="AbstractRecord" id="AbstractRecord"
                 type="csw:AbstractRecordType" abstract="true" />
diff --git ogc/filter/1.0.0/filter.xsd ogc/filter/1.0.0/filter.xsd
index f31a3b0..3a1394c 100644
--- ogc/filter/1.0.0/filter.xsd
+++ ogc/filter/1.0.0/filter.xsd
@@ -6,7 +6,7 @@
     To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<xsd:include schemaLocation="expr.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/geometry.xsd"/>
+	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/geometry.xsd"/>
 	<!-- ============================================= -->
 	<!-- FILTER EXPRESSION                             -->
 	<!-- ============================================= -->
diff --git ogc/filter/1.1.0/filter.xsd ogc/filter/1.1.0/filter.xsd
index 0eb97d2..5708362 100644
--- ogc/filter/1.1.0/filter.xsd
+++ ogc/filter/1.1.0/filter.xsd
@@ -26,7 +26,7 @@
    <xsd:include schemaLocation="filterCapabilities.xsd"/>
 
    <xsd:import namespace="http://www.opengis.net/gml"
-               schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/geometryAggregates.xsd"/>
+               schemaLocation="../../gml/3.1.1/base/geometryAggregates.xsd"/>
 
    <xsd:element name="_Id" type="ogc:AbstractIdType" abstract="true"/>
    <xsd:element name="FeatureId"
diff --git ogc/filter/2.0/filterCapabilities.xsd ogc/filter/2.0/filterCapabilities.xsd
index ac51045..287da1d 100644
--- ogc/filter/2.0/filterCapabilities.xsd
+++ ogc/filter/2.0/filterCapabilities.xsd
@@ -23,7 +23,7 @@
                schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 
    <xsd:import namespace="http://www.opengis.net/ows/1.1"
-        schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+               schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 
    <xsd:element name="Filter_Capabilities">
       <xsd:complexType>
diff --git ogc/gml/3.1.1/base/defaultStyle.xsd ogc/gml/3.1.1/base/defaultStyle.xsd
index a5ebeb7..0a0899f 100644
--- ogc/gml/3.1.1/base/defaultStyle.xsd
+++ ogc/gml/3.1.1/base/defaultStyle.xsd
@@ -15,7 +15,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
diff --git ogc/gml/3.1.1/base/temporal.xsd ogc/gml/3.1.1/base/temporal.xsd
index 96be3f7..19ef923 100644
--- ogc/gml/3.1.1/base/temporal.xsd
+++ ogc/gml/3.1.1/base/temporal.xsd
@@ -325,6 +325,16 @@
 			<enumeration value="unknown"/>
 		</restriction>
 	</simpleType>
+	<!-- ===================================== -->
+	<simpleType name="TimeDurationType">
+		<annotation>
+			<documentation xml:lang="en">
+      Base type for describing temporal length or distance. The value space is further 
+      constrained by subtypes that conform to the ISO 8601 or ISO 11404 standards.
+      </documentation>
+		</annotation>
+		<union memberTypes="duration decimal"/>
+	</simpleType>
 	<!-- ================================================================== -->
 	<!-- ==== Convenience properties ==== -->
 	<!-- ================================================================== -->
diff --git ogc/gml/3.2.1/coordinateOperations.xsd ogc/gml/3.2.1/coordinateOperations.xsd
index 5eaacaf..7f03e51 100644
--- ogc/gml/3.2.1/coordinateOperations.xsd
+++ ogc/gml/3.2.1/coordinateOperations.xsd
@@ -19,7 +19,7 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="referenceSystems.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="../../iso/19139/20070417/gmd/gmd.xsd"/>
 	<element name="AbstractCoordinateOperation" type="gml:AbstractCoordinateOperationType" abstract="true" substitutionGroup="gml:Definition">
 		<annotation>
 			<documentation>gml:AbstractCoordinateOperation is a mathematical operation on coordinates that transforms or converts coordinates to another coordinate reference system. Many but not all coordinate operations (from CRS A to CRS B) also uniquely define the inverse operation (from CRS B to CRS A). In some cases, the operation method algorithm for the inverse operation is the same as for the forward algorithm, but the signs of some operation parameter values shall be reversed. In other cases, different algorithms are required for the forward and inverse operations, but the same operation parameter values are used. If (some) entirely different parameter values are needed, a different coordinate operation shall be defined.
diff --git ogc/gml/3.2.1/defaultStyle.xsd ogc/gml/3.2.1/defaultStyle.xsd
index 3a4635d..8419f54 100644
--- ogc/gml/3.2.1/defaultStyle.xsd
+++ ogc/gml/3.2.1/defaultStyle.xsd
@@ -13,7 +13,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../../gml/3.1.1/smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
diff --git ogc/gml/3.2.1/referenceSystems.xsd ogc/gml/3.2.1/referenceSystems.xsd
index fb7e897..2d701a8 100644
--- ogc/gml/3.2.1/referenceSystems.xsd
+++ ogc/gml/3.2.1/referenceSystems.xsd
@@ -15,7 +15,7 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	</annotation>
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="dictionary.xsd"/>
-	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="../../iso/19139/20070417/gmd/gmd.xsd"/>
 	<complexType name="IdentifiedObjectType" abstract="true">
 		<annotation>
 			<documentation>gml:IdentifiedObjectType provides identification properties of a CRS-related object. In gml:DefinitionType, the gml:identifier element shall be the primary name by which this object is identified, encoding the "name" attribute in the UML model.
diff --git ogc/kml/2.2.0/ogckml22.xsd ogc/kml/2.2.0/ogckml22.xsd
index d1e6faa..0797a56 100644
--- ogc/kml/2.2.0/ogckml22.xsd
+++ ogc/kml/2.2.0/ogckml22.xsd
@@ -16,11 +16,11 @@
 
   <!-- import atom:author and atom:link -->
   <import namespace="http://www.w3.org/2005/Atom"
-    schemaLocation="atom-author-link.xsd"/>
+    schemaLocation="../../../w3c/2005/atom-author-link.xsd"/>
 
   <!-- import xAL:Address -->
   <import namespace="urn:oasis:names:tc:ciq:xsdschema:xAL:2.0"
-    schemaLocation="http://docs.oasis-open.org/election/external/xAL.xsd"/>
+    schemaLocation="../../../oasis/election/external/xAL.xsd"/>
 
   <!-- KML field types (simple content) -->
 
diff --git ogc/om/1.0.0/observation.xsd ogc/om/1.0.0/observation.xsd
index bb69963..4349ab8 100644
--- ogc/om/1.0.0/observation.xsd
+++ ogc/om/1.0.0/observation.xsd
@@ -14,12 +14,12 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	<include schemaLocation="om.xsd"/>
 	<!-- ====================================================================== -->
 	<!-- bring in other schemas -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- can't use ISO Metadata schema with GML 3.1.1
 		<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
 	-->
-	<import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
-	<import namespace="http://www.opengis.net/sensorML/1.0.1" schemaLocation="http://schemas.opengis.net/sensorML/1.0.1/sensorML.xsd"/>
+	<import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
+	<import namespace="http://www.opengis.net/sensorML/1.0.1" schemaLocation="../../sensorML/1.0.1/sensorML.xsd"/>
 	<!-- ====================================================================== -->
 	<!-- ====================================================================== -->
 	<!-- ===== Object types for Observations ====== -->
diff --git ogc/ows/2.0/owsAdditionalParameters.xsd ogc/ows/2.0/owsAdditionalParameters.xsd
index 8e91265..1f534a3 100644
--- ogc/ows/2.0/owsAdditionalParameters.xsd
+++ ogc/ows/2.0/owsAdditionalParameters.xsd
@@ -53,13 +53,13 @@
   <!-- ========================================================== -->
   <complexType name="AdditionalParametersType">
     <complexContent>
-      <extension base="ows:AdditionalParametersBaseType">
+      <restriction base="ows:AdditionalParametersBaseType">
         <sequence>
           <element ref="ows:AdditionalParameter"
                    minOccurs="0"
                    maxOccurs="unbounded" />
         </sequence>
-      </extension>
+      </restriction>
     </complexContent>
   </complexType>
   <!-- ========================================================== -->
diff --git ogc/ows/2.0/owsCommon.xsd ogc/ows/2.0/owsCommon.xsd
index 0139647..9bd628e 100644
--- ogc/ows/2.0/owsCommon.xsd
+++ ogc/ows/2.0/owsCommon.xsd
@@ -71,7 +71,7 @@
     </annotation>
     <sequence>
       <element ref="ows:AbstractMetaData"
-               minOccurs="0" />
+               minOccurs="0" maxOccurs="unbounded"/>
     </sequence>
     <attributeGroup ref="xlink:simpleAttrs">
       <annotation>
diff --git ogc/sensorML/1.0.1/base.xsd ogc/sensorML/1.0.1/base.xsd
index 85a0a3c..60913ee 100644
--- ogc/sensorML/1.0.1/base.xsd
+++ ogc/sensorML/1.0.1/base.xsd
@@ -11,11 +11,11 @@
 	</xs:annotation>
 	<!-- ================================================= -->
 	<!-- import GML 3.1.1 -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- import SWE Common Data Types 1.0.1 -->
-	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
+	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
 	<!-- import US Intelligence Community schema for security speciifications-->
-	<xs:import namespace="urn:us:gov:ic:ism:v2" schemaLocation="http://schemas.opengis.net/ic/2.0/IC-ISM-v2.xsd"/>
+	<xs:import namespace="urn:us:gov:ic:ism:v2" schemaLocation="../../ic/2.0/IC-ISM-v2.xsd"/>
 	<xs:include schemaLocation="./sensorML.xsd"/>
 	<!-- ================================================= -->
 	<xs:complexType name="AbstractSMLType" abstract="true">
diff --git ogc/sensorML/1.0.1/method.xsd ogc/sensorML/1.0.1/method.xsd
index 6840f0f..c17a3aa 100644
--- ogc/sensorML/1.0.1/method.xsd
+++ ogc/sensorML/1.0.1/method.xsd
@@ -9,8 +9,8 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
 	<xs:include schemaLocation="./sensorML.xsd"/>
 	<xs:include schemaLocation="./base.xsd"/>
 	<xs:include schemaLocation="./process.xsd"/>
diff --git ogc/sensorML/1.0.1/process.xsd ogc/sensorML/1.0.1/process.xsd
index b5fca95..793ddd5 100644
--- ogc/sensorML/1.0.1/process.xsd
+++ ogc/sensorML/1.0.1/process.xsd
@@ -9,8 +9,8 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
 	<xs:include schemaLocation="./sensorML.xsd"/>
 	<xs:include schemaLocation="./method.xsd"/>
 	<!-- ================================================= -->
diff --git ogc/sensorML/1.0.1/sensorML.xsd ogc/sensorML/1.0.1/sensorML.xsd
index 3370676..c96aa60 100644
--- ogc/sensorML/1.0.1/sensorML.xsd
+++ ogc/sensorML/1.0.1/sensorML.xsd
@@ -9,8 +9,8 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
 	<xs:include schemaLocation="./system.xsd"/>
 	<!-- ================================================= -->
 	<xs:element name="SensorML">
diff --git ogc/sensorML/1.0.1/system.xsd ogc/sensorML/1.0.1/system.xsd
index 7811316..c807258 100644
--- ogc/sensorML/1.0.1/system.xsd
+++ ogc/sensorML/1.0.1/system.xsd
@@ -9,8 +9,8 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
 	<xs:include schemaLocation="./sensorML.xsd"/>
 	<xs:include schemaLocation="./process.xsd"/>
 	<!-- ================================================= -->
diff --git ogc/sld/1.0.0/StyledLayerDescriptor.xsd ogc/sld/1.0.0/StyledLayerDescriptor.xsd
index 5a80fcd..84502fb 100644
--- ogc/sld/1.0.0/StyledLayerDescriptor.xsd
+++ ogc/sld/1.0.0/StyledLayerDescriptor.xsd
@@ -2,7 +2,7 @@
 <xsd:schema targetNamespace="http://www.opengis.net/sld" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ogc="http://www.opengis.net/ogc" xmlns:sld="http://www.opengis.net/sld" elementFormDefault="qualified" version="1.0.0.2">
 	
 	<xsd:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+	<xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 	<!-- *********************************************************************** -->
 	<xsd:annotation>
 		<xsd:documentation>
diff --git ogc/sos/1.0.0/ogc4sos.xsd ogc/sos/1.0.0/ogc4sos.xsd
index 7727b83..f2b5d66 100644
--- ogc/sos/1.0.0/ogc4sos.xsd
+++ ogc/sos/1.0.0/ogc4sos.xsd
@@ -13,8 +13,8 @@
   <!-- ==============================================================
        includes and imports
        ============================================================== -->
-  <xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-  <xs:include schemaLocation="http://schemas.opengis.net/filter/1.1.0/filter.xsd"/>
+  <xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+  <xs:include schemaLocation="../../filter/1.1.0/filter.xsd"/>
   <!-- =================================================================== -->
   <!-- TEMPORAL OPERATORS                                                  -->
   <!-- =================================================================== -->
diff --git ogc/sos/1.0.0/sosContents.xsd ogc/sos/1.0.0/sosContents.xsd
index 3b11808..2d6d0be 100644
--- ogc/sos/1.0.0/sosContents.xsd
+++ ogc/sos/1.0.0/sosContents.xsd
@@ -15,9 +15,9 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="http://schemas.opengis.net/sweCommon/1.0.1/swe.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/swe/1.0.1" schemaLocation="../../sweCommon/1.0.1/swe.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosDescribeSensor.xsd ogc/sos/1.0.0/sosDescribeSensor.xsd
index 5bbda7f..3cadf68 100644
--- ogc/sos/1.0.0/sosDescribeSensor.xsd
+++ ogc/sos/1.0.0/sosDescribeSensor.xsd
@@ -15,7 +15,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosGetCapabilities.xsd ogc/sos/1.0.0/sosGetCapabilities.xsd
index 291df65..84948b1 100644
--- ogc/sos/1.0.0/sosGetCapabilities.xsd
+++ ogc/sos/1.0.0/sosGetCapabilities.xsd
@@ -17,8 +17,8 @@
 	============================================================== -->
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosContents.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/sos/1.0.0/ogc4sos.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ogc" schemaLocation="../../sos/1.0.0/ogc4sos.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/sos/1.0.0/sosGetFeatureOfInterest.xsd ogc/sos/1.0.0/sosGetFeatureOfInterest.xsd
index 13941dc..3213296 100644
--- ogc/sos/1.0.0/sosGetFeatureOfInterest.xsd
+++ ogc/sos/1.0.0/sosGetFeatureOfInterest.xsd
@@ -15,7 +15,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/sos/1.0.0/ogc4sos.xsd"/>
+	<import namespace="http://www.opengis.net/ogc" schemaLocation="../../sos/1.0.0/ogc4sos.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosGetFeatureOfInterestTime.xsd ogc/sos/1.0.0/sosGetFeatureOfInterestTime.xsd
index 21c1978..54c766c 100644
--- ogc/sos/1.0.0/sosGetFeatureOfInterestTime.xsd
+++ ogc/sos/1.0.0/sosGetFeatureOfInterestTime.xsd
@@ -16,7 +16,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/sos/1.0.0/ogc4sos.xsd"/>
+	<import namespace="http://www.opengis.net/ogc" schemaLocation="../../sos/1.0.0/ogc4sos.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosGetObservation.xsd ogc/sos/1.0.0/sosGetObservation.xsd
index e792276..0d5f8e5 100644
--- ogc/sos/1.0.0/sosGetObservation.xsd
+++ ogc/sos/1.0.0/sosGetObservation.xsd
@@ -15,9 +15,9 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/sos/1.0.0/ogc4sos.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ogc" schemaLocation="../../sos/1.0.0/ogc4sos.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosGetObservationById.xsd ogc/sos/1.0.0/sosGetObservationById.xsd
index faec131..d6b66bd 100644
--- ogc/sos/1.0.0/sosGetObservationById.xsd
+++ ogc/sos/1.0.0/sosGetObservationById.xsd
@@ -15,8 +15,8 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/sos/1.0.0/ogc4sos.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ogc" schemaLocation="../../sos/1.0.0/ogc4sos.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosGetResult.xsd ogc/sos/1.0.0/sosGetResult.xsd
index de2bd52..a4469eb 100644
--- ogc/sos/1.0.0/sosGetResult.xsd
+++ ogc/sos/1.0.0/sosGetResult.xsd
@@ -15,7 +15,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/sos/1.0.0/ogc4sos.xsd"/>
+	<import namespace="http://www.opengis.net/ogc" schemaLocation="../../sos/1.0.0/ogc4sos.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosInsert.xsd ogc/sos/1.0.0/sosInsert.xsd
index ad8792e..c935ad3 100644
--- ogc/sos/1.0.0/sosInsert.xsd
+++ ogc/sos/1.0.0/sosInsert.xsd
@@ -16,8 +16,8 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
-	<import namespace="http://www.opengis.net/om/1.0" schemaLocation="http://schemas.opengis.net/om/1.0.0/om.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/om/1.0" schemaLocation="../../om/1.0.0/om.xsd"/>
 	<include schemaLocation="sosAll.xsd"/>
 	<include schemaLocation="sosCommon.xsd"/>
 	<!-- ==============================================================
diff --git ogc/sos/1.0.0/sosRegisterSensor.xsd ogc/sos/1.0.0/sosRegisterSensor.xsd
index cfc1fca..84b9848 100644
--- ogc/sos/1.0.0/sosRegisterSensor.xsd
+++ ogc/sos/1.0.0/sosRegisterSensor.xsd
@@ -15,8 +15,8 @@
   <!-- ==============================================================
 		includes and imports
 	============================================================== -->
-  <import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
-  <import namespace="http://www.opengis.net/om/1.0" schemaLocation="http://schemas.opengis.net/om/1.0.0/om.xsd"/>
+  <import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
+  <import namespace="http://www.opengis.net/om/1.0" schemaLocation="../../om/1.0.0/om.xsd"/>
   <include schemaLocation="sosAll.xsd"/>
   <include schemaLocation="sosCommon.xsd"/>
   <!-- ==============================================================
diff --git ogc/sweCommon/1.0.1/aggregateTypes.xsd ogc/sweCommon/1.0.1/aggregateTypes.xsd
index cc0a7a1..5b91f64 100644
--- ogc/sweCommon/1.0.1/aggregateTypes.xsd
+++ ogc/sweCommon/1.0.1/aggregateTypes.xsd
@@ -9,7 +9,7 @@
 	</xs:annotation>
 	<!-- ================================================= -->
 	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<xs:include schemaLocation="./simpleTypes.xsd"/>
 	<xs:include schemaLocation="./encoding.xsd"/>
diff --git ogc/sweCommon/1.0.1/basicTypes.xsd ogc/sweCommon/1.0.1/basicTypes.xsd
index 2645253..c726f41 100644
--- ogc/sweCommon/1.0.1/basicTypes.xsd
+++ ogc/sweCommon/1.0.1/basicTypes.xsd
@@ -10,7 +10,7 @@
 	</xs:annotation>
 	<!-- ====================================================================== -->
 	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<!-- ====================================================================== -->
 	<!-- === Units of Measure === -->
diff --git ogc/sweCommon/1.0.1/curveTypes.xsd ogc/sweCommon/1.0.1/curveTypes.xsd
index 2fd9e62..2457ba9 100644
--- ogc/sweCommon/1.0.1/curveTypes.xsd
+++ ogc/sweCommon/1.0.1/curveTypes.xsd
@@ -8,7 +8,7 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<xs:include schemaLocation="./aggregateTypes.xsd"/>
 	<!-- ================================================= -->
diff --git ogc/sweCommon/1.0.1/data.xsd ogc/sweCommon/1.0.1/data.xsd
index 6cea4c1..15c9504 100644
--- ogc/sweCommon/1.0.1/data.xsd
+++ ogc/sweCommon/1.0.1/data.xsd
@@ -8,7 +8,7 @@
 		</xs:documentation>
 	</xs:annotation>
    <!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<xs:include schemaLocation="./aggregateTypes.xsd"/>
    <!-- ================================================= -->
diff --git ogc/sweCommon/1.0.1/encoding.xsd ogc/sweCommon/1.0.1/encoding.xsd
index 31d4a0a..6818cf8 100644
--- ogc/sweCommon/1.0.1/encoding.xsd
+++ ogc/sweCommon/1.0.1/encoding.xsd
@@ -8,7 +8,7 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<!-- ================================================= -->
 	<xs:complexType name="BlockEncodingPropertyType">
diff --git ogc/sweCommon/1.0.1/phenomenon.xsd ogc/sweCommon/1.0.1/phenomenon.xsd
index 2126692..a6e6cee 100644
--- ogc/sweCommon/1.0.1/phenomenon.xsd
+++ ogc/sweCommon/1.0.1/phenomenon.xsd
@@ -12,7 +12,7 @@
 	</annotation>
 	<!-- ====================================================================== -->
 	<!-- bring in other schemas -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="./swe.xsd"/>
 	<include schemaLocation="./aggregateTypes.xsd"/>
 	<!-- ====================================================================== -->
diff --git ogc/sweCommon/1.0.1/positionTypes.xsd ogc/sweCommon/1.0.1/positionTypes.xsd
index 037236b..c57a2a0 100644
--- ogc/sweCommon/1.0.1/positionTypes.xsd
+++ ogc/sweCommon/1.0.1/positionTypes.xsd
@@ -8,7 +8,7 @@
 		</xs:documentation>
 	</xs:annotation>
 	<!-- ================================================= -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<xs:include schemaLocation="./aggregateTypes.xsd"/>
 	<!-- ================================================= -->
diff --git ogc/sweCommon/1.0.1/simpleTypes.xsd ogc/sweCommon/1.0.1/simpleTypes.xsd
index e462af3..8e27561 100644
--- ogc/sweCommon/1.0.1/simpleTypes.xsd
+++ ogc/sweCommon/1.0.1/simpleTypes.xsd
@@ -15,7 +15,7 @@ Note: In general, content model definition using composition is more easily acco
 	</xs:annotation>
 	<!-- ================================================= -->
 	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<xs:include schemaLocation="./swe.xsd"/>
 	<xs:include schemaLocation="./basicTypes.xsd"/>
 	<!-- ================================================= -->
diff --git ogc/sweCommon/1.0.1/temporalAggregates.xsd ogc/sweCommon/1.0.1/temporalAggregates.xsd
index 093df59..bf9b952 100644
--- ogc/sweCommon/1.0.1/temporalAggregates.xsd
+++ ogc/sweCommon/1.0.1/temporalAggregates.xsd
@@ -11,7 +11,7 @@
 	</annotation>
 	<!-- ====================================================================== -->
 	<!-- bring in other schemas -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="./swe.xsd"/>
 	<!-- ================================================================== -->
 	<!-- =========================================================== -->
diff --git ogc/wcs/1.0.0/describeCoverage.xsd ogc/wcs/1.0.0/describeCoverage.xsd
index 0a544c4..d1ab75d 100644
--- ogc/wcs/1.0.0/describeCoverage.xsd
+++ ogc/wcs/1.0.0/describeCoverage.xsd
@@ -41,7 +41,7 @@
 	<xs:include schemaLocation="./wcsAll.xsd"/>
 	<xs:include schemaLocation="./wcsCapabilities.xsd"/>
 	<xs:include schemaLocation="./values.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.0.0/getCoverage.xsd ogc/wcs/1.0.0/getCoverage.xsd
index 94d4d17..84776f1 100644
--- ogc/wcs/1.0.0/getCoverage.xsd
+++ ogc/wcs/1.0.0/getCoverage.xsd
@@ -40,7 +40,7 @@
 	========================================================== -->
 	<xs:include schemaLocation="./wcsAll.xsd"/>
 	<xs:include schemaLocation="./describeCoverage.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.0.0/owsBase.xsd ogc/wcs/1.0.0/owsBase.xsd
index 6410894..cf9769a 100644
--- ogc/wcs/1.0.0/owsBase.xsd
+++ ogc/wcs/1.0.0/owsBase.xsd
@@ -38,7 +38,7 @@
 	<!-- ==============================================================
            includes and imports
 	============================================================== -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ============================================================
 		 Elements and types		
 	============================================================== -->
diff --git ogc/wcs/1.0.0/wcsCapabilities.xsd ogc/wcs/1.0.0/wcsCapabilities.xsd
index 5997a13..ade4e0c 100644
--- ogc/wcs/1.0.0/wcsCapabilities.xsd
+++ ogc/wcs/1.0.0/wcsCapabilities.xsd
@@ -45,7 +45,7 @@
     <!-- ========= includes and imports ================================ -->
     <xs:include schemaLocation="./wcsAll.xsd"/>
     <xs:include schemaLocation="./owsBase.xsd"/>
-    <xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+    <xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
     <xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
     <!-- ==============================================================       elements and types	============================================================== -->
     <xs:element name="GetCapabilities">
diff --git ogc/wcs/1.1/wcsCommon.xsd ogc/wcs/1.1/wcsCommon.xsd
index fe35a4c..645ae30 100644
--- ogc/wcs/1.1/wcsCommon.xsd
+++ ogc/wcs/1.1/wcsCommon.xsd
@@ -20,7 +20,7 @@ version="1.1.3">
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================		
 	elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsContents.xsd ogc/wcs/1.1/wcsContents.xsd
index d58b9e4..578de13 100644
--- ogc/wcs/1.1/wcsContents.xsd
+++ ogc/wcs/1.1/wcsContents.xsd
@@ -18,7 +18,7 @@ version="1.1.3">
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsCommon.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsCoverages.xsd ogc/wcs/1.1/wcsCoverages.xsd
index d6670f7..546a6b7 100644
--- ogc/wcs/1.1/wcsCoverages.xsd
+++ ogc/wcs/1.1/wcsCoverages.xsd
@@ -17,7 +17,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==========================================================
 		Types and elements
 	    ========================================================== -->
diff --git ogc/wcs/1.1/wcsDescribeCoverage.xsd ogc/wcs/1.1/wcsDescribeCoverage.xsd
index 29c4bfe..d026fd2 100644
--- ogc/wcs/1.1/wcsDescribeCoverage.xsd
+++ ogc/wcs/1.1/wcsDescribeCoverage.xsd
@@ -24,8 +24,8 @@ version="1.1.3">
 	<include schemaLocation="wcsCommon.xsd"></include>
 	<include schemaLocation="wcsGridCRS.xsd"></include>
 	<include schemaLocation="wcsInterpolationMethod.xsd"></include>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"></import>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"></import>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"></import>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"></import>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGetCapabilities.xsd ogc/wcs/1.1/wcsGetCapabilities.xsd
index c439f2d..043867e 100644
--- ogc/wcs/1.1/wcsGetCapabilities.xsd
+++ ogc/wcs/1.1/wcsGetCapabilities.xsd
@@ -18,7 +18,7 @@ version="1.1.3">
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsContents.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGetCoverage.xsd ogc/wcs/1.1/wcsGetCoverage.xsd
index 5cca02a..bb9eb43 100644
--- ogc/wcs/1.1/wcsGetCoverage.xsd
+++ ogc/wcs/1.1/wcsGetCoverage.xsd
@@ -20,7 +20,7 @@
 	<include schemaLocation="wcsCommon.xsd"/>
 	<include schemaLocation="wcsGridCRS.xsd"/>
 	<include schemaLocation="wcsCoverages.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================	
 		elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGridCRS.xsd ogc/wcs/1.1/wcsGridCRS.xsd
index 18d96b7..b5c6e97 100644
--- ogc/wcs/1.1/wcsGridCRS.xsd
+++ ogc/wcs/1.1/wcsGridCRS.xsd
@@ -21,7 +21,7 @@
 	<!-- ========================================================
            includes and imports
 	======================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ========================================================
 		elements and types
 	======================================================== -->
diff --git ogc/wcs/1.1/wcsInterpolationMethod.xsd ogc/wcs/1.1/wcsInterpolationMethod.xsd
index c6288d0..3a6b829 100644
--- ogc/wcs/1.1/wcsInterpolationMethod.xsd
+++ ogc/wcs/1.1/wcsInterpolationMethod.xsd
@@ -17,7 +17,7 @@ version="1.1.3">
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff --git ogc/wfs/1.0.0/WFS-basic.xsd ogc/wfs/1.0.0/WFS-basic.xsd
index 4797ba3..ddfe0c8 100644
--- ogc/wfs/1.0.0/WFS-basic.xsd
+++ ogc/wfs/1.0.0/WFS-basic.xsd
@@ -19,8 +19,8 @@
         Includes and Imports
         ============================================================== -->
    <xsd:include schemaLocation="./wfs.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/feature.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 
    <!-- ==============================================================
         REQUEST MESSAGES 
diff --git ogc/wfs/1.0.0/WFS-capabilities.xsd ogc/wfs/1.0.0/WFS-capabilities.xsd
index b408876..45c00b5 100644
--- ogc/wfs/1.0.0/WFS-capabilities.xsd
+++ ogc/wfs/1.0.0/WFS-capabilities.xsd
@@ -24,7 +24,7 @@
    <!-- ==============================================================
         Imports ...
         ============================================================== -->
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filterCapabilities.xsd" />
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filterCapabilities.xsd" />
 
    <!-- ==============================================================
         Global elements and attributes
diff --git ogc/wfs/1.0.0/WFS-transaction.xsd ogc/wfs/1.0.0/WFS-transaction.xsd
index 0cbe0c7..5072dd4 100644
--- ogc/wfs/1.0.0/WFS-transaction.xsd
+++ ogc/wfs/1.0.0/WFS-transaction.xsd
@@ -18,8 +18,8 @@
         Includes and Imports
         ============================================================== -->
    <xsd:include schemaLocation="./wfs.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/feature.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 
    <!-- ==============================================================
          REQUEST MESSAGES
diff --git ogc/wfs/1.1.0/wfs.xsd ogc/wfs/1.1.0/wfs.xsd
index 1b90d26..87ac05e 100644
--- ogc/wfs/1.1.0/wfs.xsd
+++ ogc/wfs/1.1.0/wfs.xsd
@@ -18,9 +18,9 @@
    <!-- ==============================================================
         Includes and Imports
         ============================================================== -->
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.1.0/filter.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.1.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
    <!-- ================================================================== -->
    <!-- =   BASE REQUEST TYPE                                            = -->
diff --git ogc/wfs/2.0/wfs.xsd ogc/wfs/2.0/wfs.xsd
index a108afa..2f63bee 100644
--- ogc/wfs/2.0/wfs.xsd
+++ ogc/wfs/2.0/wfs.xsd
@@ -25,9 +25,9 @@
    <xsd:import namespace="http://www.w3.org/1999/xlink"
       schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
    <xsd:import namespace="http://www.opengis.net/ows/1.1"
-      schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+      schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
    <xsd:import namespace="http://www.opengis.net/fes/2.0"
-      schemaLocation="http://schemas.opengis.net/filter/2.0/filterAll.xsd"/>
+      schemaLocation="../../filter/2.0/filterAll.xsd"/>
    <!-- ================================================================== -->
    <!-- =   BASE REQUEST TYPE                                            = -->
    <!-- ================================================================== -->
diff --git ogc/wps/1.0.0/common/DescriptionType.xsd ogc/wps/1.0.0/common/DescriptionType.xsd
index 7413109..ba87d64 100644
--- ogc/wps/1.0.0/common/DescriptionType.xsd
+++ ogc/wps/1.0.0/common/DescriptionType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/common/ProcessBriefType.xsd ogc/wps/1.0.0/common/ProcessBriefType.xsd
index e783a43..2303c3a 100644
--- ogc/wps/1.0.0/common/ProcessBriefType.xsd
+++ ogc/wps/1.0.0/common/ProcessBriefType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="DescriptionType.xsd"/>
 	<include schemaLocation="ProcessVersion.xsd"/>
 	<include schemaLocation="WSDL.xsd"/>
diff --git ogc/wps/1.0.0/common/RequestBaseType.xsd ogc/wps/1.0.0/common/RequestBaseType.xsd
index 071b0df..c98294b 100644
--- ogc/wps/1.0.0/common/RequestBaseType.xsd
+++ ogc/wps/1.0.0/common/RequestBaseType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/common/ResponseBaseType.xsd ogc/wps/1.0.0/common/ResponseBaseType.xsd
index b721bc6..0f5a95e 100644
--- ogc/wps/1.0.0/common/ResponseBaseType.xsd
+++ ogc/wps/1.0.0/common/ResponseBaseType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 	<!-- ==============================================================
 		elements and types
diff --git ogc/wps/1.0.0/wpsDescribeProcess_request.xsd ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
index 0f5e37a..2df7127 100644
--- ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
+++ ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
@@ -14,7 +14,7 @@
 	<!-- =============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/RequestBaseType.xsd"/>
 	<!-- =============================================================
diff --git ogc/wps/1.0.0/wpsDescribeProcess_response.xsd ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
index 4cdb7a7..dfb7e86 100644
--- ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
+++ ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
@@ -18,7 +18,7 @@
 	<include schemaLocation="common/ResponseBaseType.xsd"/>
 	<include schemaLocation="common/DescriptionType.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsExecute_request.xsd ogc/wps/1.0.0/wpsExecute_request.xsd
index 38f137f..b75ca69 100644
--- ogc/wps/1.0.0/wpsExecute_request.xsd
+++ ogc/wps/1.0.0/wpsExecute_request.xsd
@@ -16,7 +16,7 @@
 	============================================================== -->
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/RequestBaseType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<!-- =============================================================
 		elements and types
diff --git ogc/wps/1.0.0/wpsExecute_response.xsd ogc/wps/1.0.0/wpsExecute_response.xsd
index eade869..f407d64 100644
--- ogc/wps/1.0.0/wpsExecute_response.xsd
+++ ogc/wps/1.0.0/wpsExecute_response.xsd
@@ -18,7 +18,7 @@
 	<include schemaLocation="common/ResponseBaseType.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
 	<include schemaLocation="wpsExecute_request.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsGetCapabilities_request.xsd ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
index 718b250..c278d76 100644
--- ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
+++ ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
@@ -15,7 +15,7 @@
 		includes and imports
 	============================================================== -->
 	<include schemaLocation="wpsAll.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsGetCapabilities_response.xsd ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
index 300ba15..b2de275 100644
--- ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
+++ ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
@@ -17,7 +17,7 @@
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/WSDL.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 	<!-- ==============================================================
