diff --git ogc/ows/2.0/owsAdditionalParameters.xsd ogc/ows/2.0/owsAdditionalParameters.xsd
index 8e91265..1f534a3 100644
--- ogc/ows/2.0/owsAdditionalParameters.xsd
+++ ogc/ows/2.0/owsAdditionalParameters.xsd
@@ -53,13 +53,13 @@
   <!-- ========================================================== -->
   <complexType name="AdditionalParametersType">
     <complexContent>
-      <extension base="ows:AdditionalParametersBaseType">
+      <restriction base="ows:AdditionalParametersBaseType">
         <sequence>
           <element ref="ows:AdditionalParameter"
                    minOccurs="0"
                    maxOccurs="unbounded" />
         </sequence>
-      </extension>
+      </restriction>
     </complexContent>
   </complexType>
   <!-- ========================================================== -->
