diff --git ogc/csw/2.0.2/CSW-discovery.xsd ogc/csw/2.0.2/CSW-discovery.xsd
index ab0f10b..5c8164b 100644
--- ogc/csw/2.0.2/CSW-discovery.xsd
+++ ogc/csw/2.0.2/CSW-discovery.xsd
@@ -25,9 +25,9 @@
    <xsd:include schemaLocation="csw.xsd"/>
    <xsd:include schemaLocation="record.xsd"/>
 
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.1.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.1.0/filter.xsd"/>
 
    <xsd:complexType name="RequestBaseType" id="RequestBaseType" abstract="true">
       <xsd:annotation>
diff --git ogc/csw/2.0.2/rec-dcterms.xsd ogc/csw/2.0.2/rec-dcterms.xsd
index f2450e3..d4f1ae6 100644
--- ogc/csw/2.0.2/rec-dcterms.xsd
+++ ogc/csw/2.0.2/rec-dcterms.xsd
@@ -9,7 +9,7 @@
          source="http://dublincore.org/documents/dcmi-terms/">This schema declares additional DCMI elements and element refinements 
     in the "http://purl.org/dc/terms/" namespace.</xs:documentation>
    </xs:annotation>
-   <xs:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="http://schemas.opengis.net/csw/2.0.2/rec-dcmes.xsd"/>
+   <xs:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="../../csw/2.0.2/rec-dcmes.xsd"/>
    <xs:element name="abstract" type="dc:SimpleLiteral"
       substitutionGroup="dc:description"/>
    <xs:element name="accessRights" type="dc:SimpleLiteral"
diff --git ogc/csw/2.0.2/record.xsd ogc/csw/2.0.2/record.xsd
index 6919b02..0be90d6 100644
--- ogc/csw/2.0.2/record.xsd
+++ ogc/csw/2.0.2/record.xsd
@@ -22,9 +22,9 @@
    </xsd:annotation>
 
    <xsd:include schemaLocation="csw.xsd"/>
-   <xsd:import namespace="http://purl.org/dc/terms/" schemaLocation="http://schemas.opengis.net/csw/2.0.2/rec-dcterms.xsd"/>
-   <xsd:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="http://schemas.opengis.net/csw/2.0.2/rec-dcmes.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://purl.org/dc/terms/" schemaLocation="../../csw/2.0.2/rec-dcterms.xsd"/>
+   <xsd:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="../../csw/2.0.2/rec-dcmes.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
    <xsd:element name="AbstractRecord" id="AbstractRecord"
                 type="csw:AbstractRecordType" abstract="true" />
diff --git ogc/filter/1.0.0/filter.xsd ogc/filter/1.0.0/filter.xsd
index f31a3b0..3a1394c 100644
--- ogc/filter/1.0.0/filter.xsd
+++ ogc/filter/1.0.0/filter.xsd
@@ -6,7 +6,7 @@
     To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<xsd:include schemaLocation="expr.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/geometry.xsd"/>
+	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/geometry.xsd"/>
 	<!-- ============================================= -->
 	<!-- FILTER EXPRESSION                             -->
 	<!-- ============================================= -->
diff --git ogc/filter/1.1.0/filter.xsd ogc/filter/1.1.0/filter.xsd
index 0eb97d2..5708362 100644
--- ogc/filter/1.1.0/filter.xsd
+++ ogc/filter/1.1.0/filter.xsd
@@ -26,7 +26,7 @@
    <xsd:include schemaLocation="filterCapabilities.xsd"/>
 
    <xsd:import namespace="http://www.opengis.net/gml"
-               schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/geometryAggregates.xsd"/>
+               schemaLocation="../../gml/3.1.1/base/geometryAggregates.xsd"/>
 
    <xsd:element name="_Id" type="ogc:AbstractIdType" abstract="true"/>
    <xsd:element name="FeatureId"
diff --git ogc/filter/2.0/filterCapabilities.xsd ogc/filter/2.0/filterCapabilities.xsd
index a7368b4..c10f712 100644
--- ogc/filter/2.0/filterCapabilities.xsd
+++ ogc/filter/2.0/filterCapabilities.xsd
@@ -23,7 +23,7 @@
                schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 
    <xsd:import namespace="http://www.opengis.net/ows/1.1"
-               schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+               schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 
    <xsd:element name="Filter_Capabilities">
       <xsd:complexType>
diff --git ogc/gml/3.1.1/base/defaultStyle.xsd ogc/gml/3.1.1/base/defaultStyle.xsd
index a5ebeb7..0a0899f 100644
--- ogc/gml/3.1.1/base/defaultStyle.xsd
+++ ogc/gml/3.1.1/base/defaultStyle.xsd
@@ -15,7 +15,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
diff --git ogc/gml/3.1.1/base/temporal.xsd ogc/gml/3.1.1/base/temporal.xsd
index 96be3f7..19ef923 100644
--- ogc/gml/3.1.1/base/temporal.xsd
+++ ogc/gml/3.1.1/base/temporal.xsd
@@ -325,6 +325,16 @@
 			<enumeration value="unknown"/>
 		</restriction>
 	</simpleType>
+	<!-- ===================================== -->
+	<simpleType name="TimeDurationType">
+		<annotation>
+			<documentation xml:lang="en">
+      Base type for describing temporal length or distance. The value space is further 
+      constrained by subtypes that conform to the ISO 8601 or ISO 11404 standards.
+      </documentation>
+		</annotation>
+		<union memberTypes="duration decimal"/>
+	</simpleType>
 	<!-- ================================================================== -->
 	<!-- ==== Convenience properties ==== -->
 	<!-- ================================================================== -->
diff --git ogc/gml/3.2.1/coordinateOperations.xsd ogc/gml/3.2.1/coordinateOperations.xsd
index 5eaacaf..7f03e51 100644
--- ogc/gml/3.2.1/coordinateOperations.xsd
+++ ogc/gml/3.2.1/coordinateOperations.xsd
@@ -19,7 +19,7 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="referenceSystems.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="../../iso/19139/20070417/gmd/gmd.xsd"/>
 	<element name="AbstractCoordinateOperation" type="gml:AbstractCoordinateOperationType" abstract="true" substitutionGroup="gml:Definition">
 		<annotation>
 			<documentation>gml:AbstractCoordinateOperation is a mathematical operation on coordinates that transforms or converts coordinates to another coordinate reference system. Many but not all coordinate operations (from CRS A to CRS B) also uniquely define the inverse operation (from CRS B to CRS A). In some cases, the operation method algorithm for the inverse operation is the same as for the forward algorithm, but the signs of some operation parameter values shall be reversed. In other cases, different algorithms are required for the forward and inverse operations, but the same operation parameter values are used. If (some) entirely different parameter values are needed, a different coordinate operation shall be defined.
diff --git ogc/gml/3.2.1/defaultStyle.xsd ogc/gml/3.2.1/defaultStyle.xsd
index 3a4635d..8419f54 100644
--- ogc/gml/3.2.1/defaultStyle.xsd
+++ ogc/gml/3.2.1/defaultStyle.xsd
@@ -13,7 +13,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../../gml/3.1.1/smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
diff --git ogc/gml/3.2.1/referenceSystems.xsd ogc/gml/3.2.1/referenceSystems.xsd
index fb7e897..2d701a8 100644
--- ogc/gml/3.2.1/referenceSystems.xsd
+++ ogc/gml/3.2.1/referenceSystems.xsd
@@ -15,7 +15,7 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	</annotation>
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="dictionary.xsd"/>
-	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="../../iso/19139/20070417/gmd/gmd.xsd"/>
 	<complexType name="IdentifiedObjectType" abstract="true">
 		<annotation>
 			<documentation>gml:IdentifiedObjectType provides identification properties of a CRS-related object. In gml:DefinitionType, the gml:identifier element shall be the primary name by which this object is identified, encoding the "name" attribute in the UML model.
diff --git ogc/kml/2.2.0/ogckml22.xsd ogc/kml/2.2.0/ogckml22.xsd
index d1e6faa..0797a56 100644
--- ogc/kml/2.2.0/ogckml22.xsd
+++ ogc/kml/2.2.0/ogckml22.xsd
@@ -16,11 +16,11 @@
 
   <!-- import atom:author and atom:link -->
   <import namespace="http://www.w3.org/2005/Atom"
-    schemaLocation="atom-author-link.xsd"/>
+    schemaLocation="../../../w3c/2005/atom-author-link.xsd"/>
 
   <!-- import xAL:Address -->
   <import namespace="urn:oasis:names:tc:ciq:xsdschema:xAL:2.0"
-    schemaLocation="http://docs.oasis-open.org/election/external/xAL.xsd"/>
+    schemaLocation="../../../oasis/election/external/xAL.xsd"/>
 
   <!-- KML field types (simple content) -->
 
diff --git ogc/owc/0.3.0/owsforowscontext.xsd ogc/owc/0.3.0/owsforowscontext.xsd
new file mode 100644
index 0000000..eed3bde
--- /dev/null
+++ ogc/owc/0.3.0/owsforowscontext.xsd
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema version="0.1.0" targetNamespace="http://www.opengis.net/ows" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+	<xs:annotation>
+		<xs:appinfo>owsforowscontext.xsd 2008/03/10</xs:appinfo>
+		<xs:documentation>This XML Schema is a convenience for importing OWS schema files.  Primary editor: Raj Singh.  Copyright (c) 2008 OpenGIS, All Rights Reserved.</xs:documentation>
+	</xs:annotation>
+	
+	<xs:include schemaLocation="../../ows/1.0.0/owsCommon.xsd"/>
+	<xs:include schemaLocation="../../ows/1.0.0/owsServiceProvider.xsd"/>
+	<xs:include schemaLocation="../../ows/1.0.0/owsDataIdentification.xsd"/>
+	<xs:include schemaLocation="../../ows/1.0.0/ows19115subset.xsd"/>
+</xs:schema>
\ No newline at end of file
diff --git ogc/owc/0.3.1/owsContext.xsd ogc/owc/0.3.1/owsContext.xsd
new file mode 100644
index 0000000..2903dd7
--- /dev/null
+++ ogc/owc/0.3.1/owsContext.xsd
@@ -0,0 +1,529 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<xs:schema targetNamespace="http://www.opengis.net/ows-context" elementFormDefault="qualified"
+	version="0.3.1" xmlns:owc="http://www.opengis.net/ows-context"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml"
+	xmlns:ows="http://www.opengis.net/ows" xmlns:ogc="http://www.opengis.net/ogc"
+	xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink"
+	xmlns:kml="http://www.opengis.net/kml/2.2">
+	<xs:annotation>
+		<xs:appinfo>context.xsd 2009/05/08</xs:appinfo>
+		<xs:documentation>This XML Schema defines the encoding of an OWS Context Document. Primary
+			editor: Tom Kralidis. Copyright (c) 2007 OpenGIS, All Rights Reserved.</xs:documentation>
+	</xs:annotation>
+
+	<!-- ==============================================================
+	     includes and imports
+	     ============================================================== -->
+	<xs:import namespace="http://www.w3.org/1999/xlink"
+		schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml"
+		schemaLocation="../../gml/2.1.2/feature.xsd"/>
+	<xs:import namespace="http://www.opengis.net/ogc"
+		schemaLocation="../../filter/1.0.0/filter.xsd"/>
+	<xs:import namespace="http://www.opengis.net/sld"
+		schemaLocation="../../sld/1.0.0/StyledLayerDescriptor.xsd"/>
+	<xs:import namespace="http://www.opengis.net/ows"
+		schemaLocation="../../owc/0.3.0/owsforowscontext.xsd"/>
+	<xs:import namespace="http://www.opengis.net/kml/2.2"
+		schemaLocation="../../kml/2.2.0/ogckml22.xsd"/>
+	<!-- ==============================================================
+	     elements and types
+	     ============================================================== -->
+	<xs:element name="ResourceList" type="owc:ResourceListType">
+		<xs:annotation>
+			<xs:documentation>A Context document contains one map defined within a ResourceList.
+				This element describes the list of all resources to build a required application
+				view. The Layer/FeatureType/Coverage elements are presented in a bottom to top
+				approach. Each Resource is defined in a Layer OR FeatureType OR Coverage element in
+				the Context document XML. If desired, Resources may be repeated with different
+				attributes (e.g. different styles).</xs:documentation>
+		</xs:annotation>
+	</xs:element>
+	<xs:element name="OWSContext" type="owc:OWSContextType">
+		<xs:annotation>
+			<xs:documentation>The root element of the Context, this element must contain the version
+				number of the Context Specification which the current Context document complies
+				with. This element is composed of two elements, one for general information and
+				another defining a list of resources.</xs:documentation>
+		</xs:annotation>
+	</xs:element>
+	<xs:complexType name="OWSContextType">
+		<xs:sequence>
+			<xs:element name="General" type="owc:GeneralType">
+				<xs:annotation>
+					<xs:documentation>An element "General" that contains
+						layer-independent information about the current Context document such as the
+						bounding box and the spatial reference system and also some information
+						describing the Context document itself such as title, abstract, etc. This
+						element is required.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="owc:ResourceList"/>
+		</xs:sequence>
+		<xs:attribute name="version" type="xs:string" use="required" fixed="0.3.1">
+			<xs:annotation>
+				<xs:appinfo source="An attribute &quot;version&quot; representing the published specification version number, contains three positive integers, separated by decimal points, in the form &quot;x.y.z&quot;. Each context specification is numbered independently. This attribute is required."/>
+				<xs:documentation>The version of the specification. Fixed value of 'x.y.z'.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="id" type="xs:string" use="required">
+			<xs:annotation>
+				<xs:appinfo source="An attribute &quot;id&quot; which should be unique and that represents a unique identifier of the Context. This attribute is required."/>
+				<xs:documentation>An identifier for the document. This identifier may further be
+					used to identify the document in catalogs.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+	</xs:complexType>
+	<xs:complexType name="GeneralType">
+		<xs:sequence>
+			<xs:element name="Window" type="owc:WindowType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>An element "Window" presenting the size in
+						pixels of the map the Context document describes. Negotiation between
+						Context defined aspect ratio and typical client aspect ratio (according to
+						the client’s vendor) is left to the client. This element is
+					optional.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="ows:BoundingBox"/>
+			<xs:element ref="sld:MinScaleDenominator" minOccurs="0"/>
+			<xs:element ref="sld:MaxScaleDenominator" minOccurs="0"/>
+			<xs:element ref="ows:Title" minOccurs="0"/>
+			<xs:element ref="ows:Abstract" minOccurs="0"/>
+			<xs:element ref="ows:Keywords" minOccurs="0"/>
+			<xs:element name="LogoURL" type="owc:URLType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>A reference to an image that might be attached to the Context
+						document. It can be, for instance, the logo of the project for which the
+						context has been setup, or an overview of the map the context describes.
+						This element contains a link to the image as well as the dimension of the
+						image (in pixels) and its format. This element is
+					optional.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="DescriptionURL" type="owc:URLType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>A URL reference to a webpage which contains relevant
+						information to the view. This element is optional.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="ows:ServiceProvider" minOccurs="0"/>
+			<xs:element name="Extension" type="owc:ExtensionType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The Extension element is a container tag in which arbitrary
+						vendor specific information can be included without compromising the ability
+						of other clients to enforce schema validation. This tag should not be used
+						to introduce new candidate elements that are intended to promote
+						interoperability. Content in an &lt;Extension&gt; element should not
+						be expected to be preserved in transfers of Context documents between
+						different systems.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="WindowType">
+		<xs:attribute name="width" type="xs:integer" use="required">
+			<xs:annotation>
+				<xs:documentation>The width of the Window. In dimensions pixels.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="height" type="xs:integer" use="required">
+			<xs:annotation>
+				<xs:documentation>The height of the Window, in dimensions pixels</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+	</xs:complexType>
+	<xs:complexType name="URLType">
+		<xs:sequence>
+			<xs:element name="OnlineResource" type="owc:OnlineResourceType">
+				<xs:annotation>
+					<xs:documentation>Connect point URL prefix for the HTTP request method for this
+						operation request. The xlink:href attribute in the xlink:simpleLink
+						attribute group shall be used to contain this URL prefix. The other
+						attributes in the xlink:simpleLink attribute group should not be
+					used</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+		<xs:attribute name="width" type="xs:integer" use="optional">
+			<xs:annotation>
+				<xs:appinfo source="width of the resource (typically an image)"/>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="height" type="xs:integer" use="optional">
+			<xs:annotation>
+				<xs:appinfo source="height of the resource (typically an image)"/>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="format" type="xs:string" use="optional">
+			<xs:annotation>
+				<xs:appinfo source="MIME type format of the resource"/>
+			</xs:annotation>
+		</xs:attribute>
+	</xs:complexType>
+	
+	<xs:complexType name="ResourceListType">
+		<xs:sequence>
+			<xs:element ref="owc:Layer" minOccurs="0" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:complexType>
+	
+	<xs:element name="Layer" type="owc:LayerType">
+		<xs:annotation>
+			<xs:documentation>The Layer element must enclose one or more child elements
+				providing information about the Layer. The meaning of these elements is
+				defined hereunder.</xs:documentation>
+		</xs:annotation>
+	</xs:element>
+
+	<xs:complexType name="InlineFeatureCollectionType">
+		<xs:complexContent>
+			<xs:extension base="gml:AbstractFeatureCollectionType"/>
+		</xs:complexContent>
+	</xs:complexType>
+
+	<xs:complexType name="LayerType">
+		<xs:complexContent>
+			<xs:extension base="owc:AbstractResourceType">
+				<xs:sequence>
+					<!-- useful for WMS and WCS -->
+					<xs:element name="DimensionList" type="owc:DimensionListType" minOccurs="0"/>
+					<!-- useful for WCS -->
+					<xs:element name="ResponseCRS" type="xs:string" minOccurs="0"/>
+					<xs:element name="ParameterList" type="owc:ParameterListType" minOccurs="0"/>
+					<xs:element name="Depth" type="xs:string" minOccurs="0"/>
+					<xs:element name="Resx" type="xs:string" minOccurs="0"/>
+					<xs:element name="Resy" type="xs:string" minOccurs="0"/>
+					<xs:element name="Resz" type="xs:string" minOccurs="0"/>
+					<!-- useful for WFS -->
+					<xs:element name="MaxFeatures" type="xs:integer" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Maximum features that should be used to query the
+								layer. A FeatureType can potentially contain a very large dataset
+								that is impractical to download to a client, and/or too large for a
+								client's renderer.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element ref="ogc:Filter" minOccurs="0"/>
+					<!-- needed for inline GML -->
+					<xs:element name="InlineGeometry" type="owc:InlineFeatureCollectionType"
+						minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>A GML FeatureCollection provided inline within the document.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<!-- needed for inline KML -->
+					<xs:element ref="kml:Document" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="VendorExtension" type="owc:ExtensionType" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>The Extension element is a container tag in which arbitrary
+								vendor specific information can be included without compromising the ability
+								of other clients to enforce schema validation. This tag should not be used
+								to introduce new candidate elements that are intended to promote
+								interoperability. Content in an &lt;Extension&gt; element should not
+								be expected to be preserved in transfers of Context documents between
+								different systems.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+				</xs:sequence>
+				<xs:attribute name="queryable" type="xs:boolean" use="optional" default="0">
+					<xs:annotation>
+						<xs:documentation>queryable : contains true if the layer is set queryable
+							(required for a WMS layer)"</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	<xs:complexType name="AbstractResourceType" abstract="true">
+		<xs:complexContent>
+			<xs:extension base="ows:IdentificationType">
+				<xs:sequence>
+					<xs:element name="Server" type="owc:ServerType" minOccurs="0"
+						maxOccurs="unbounded">
+						<xs:annotation>
+							<xs:documentation>The element defining the service from which the named
+								Layer may be requested</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="DataURL" type="owc:URLType" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>This element contains a link to an online resource
+								where data corresponding to the layer can be found. This element is
+								optional.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="MetadataURL" type="owc:URLType" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>This element contains a link to an online resource
+								where descriptive metadata corresponding to the layer can be found.
+								This element is optional.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element ref="sld:MinScaleDenominator" minOccurs="0"/>
+					<xs:element ref="sld:MaxScaleDenominator" minOccurs="0"/>
+					<xs:element name="StyleList" type="owc:StyleListType" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>The parent element containing the list of available
+								styles for this layer. Each style is defined in a Style element. A
+								StyleList shall include at least one Style
+							element.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Extension" type="owc:ExtensionType" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>The Extension element is a container tag in which
+								arbitrary vendor specific information can be included without
+								compromising the ability of other clients to enforce schema
+								validation. This tag should not be used to introduce new candidate
+								elements that are intended to promote interoperability. Content in
+								an &lt;Extension&gt; element should not be expected to be
+								preserved in transfers of Context documents between different
+								systems.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<!-- needed for nesting Layers -->
+					<xs:element ref="owc:Layer" minOccurs="0" maxOccurs="unbounded"/>
+				</xs:sequence>
+				<xs:attribute name="name">
+					<xs:annotation>
+						<xs:documentation>An human-readable identifier for the
+						object.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="id">
+					<xs:annotation>
+						<xs:documentation>An identifier for the document. This identifier may
+							further be used to identify the document in catalogs.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="group" use="optional">
+					<xs:annotation>
+						<xs:documentation>group: name which enables a viewer client generator to
+							group in providing grouped themes of layers/feature
+						types/coverages</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="hidden" type="xs:boolean" use="optional" default="0">
+					<xs:annotation>
+						<xs:documentation>hidden : contains true if the layer should be hidden in
+							the client result map (required)</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="opacity" type="xs:decimal" use="optional" default="1">
+					<xs:annotation>
+						<xs:documentation>opacity:0: map data represents vector features that
+							probably do not completely fill space. 1: map data are mostly or
+							completely opaque.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	<xs:complexType name="ServerType">
+		<xs:sequence>
+			<xs:element name="OnlineResource" type="owc:OnlineResourceType" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Online Resource element: the link to the online resource
+						(required)</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+		<xs:attribute name="default" type="xs:boolean" use="optional">
+			<xs:annotation>
+				<xs:documentation>Whether this is the default binding endpoint for this resource.
+					Resources may have multiple bindings (for example, as a means to associate a WMS
+					and WFS)</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="service" type="owc:serviceType" use="required">
+			<xs:annotation>
+				<xs:documentation>Service : the type of the service according to OGC interfaces,
+					such as WMS, WFS. (required)</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="version" type="xs:string" use="required">
+			<xs:annotation>
+				<xs:documentation>Version : Version number of the OGC interface specification which
+					corresponds to the service (required)</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="title" type="xs:string" use="optional">
+			<xs:annotation>
+				<xs:documentation>Title : the title of the service (extracted from the Capabilities
+					by the Context document creator) (optional)"</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+	</xs:complexType>
+	<xs:simpleType name="serviceType">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="urn:ogc:serviceType:WMS"/>
+			<xs:enumeration value="urn:ogc:serviceType:WFS"/>
+			<xs:enumeration value="urn:ogc:serviceType:WCS"/>
+			<xs:enumeration value="urn:ogc:serviceType:GML"/>
+			<xs:enumeration value="urn:ogc:serviceType:SLD"/>
+			<xs:enumeration value="urn:ogc:serviceType:FES"/>
+			<xs:enumeration value="urn:ogc:serviceType:KML"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:complexType name="FormatListType">
+		<xs:sequence>
+			<xs:element name="Format" type="owc:FormatType" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Describe one output image format for the
+					resource.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="FormatType">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="current" type="xs:boolean">
+					<xs:annotation>
+						<xs:documentation>current : contains 1 if the current image format is
+							selected."</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+			</xs:extension>
+		</xs:simpleContent>
+	</xs:complexType>
+	<xs:complexType name="StyleListType">
+		<xs:sequence>
+			<xs:element name="Style" type="owc:StyleType" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>A Style element may be composed in two different ways: named
+						style or SLD.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="StyleType">
+		<xs:choice>
+			<xs:sequence>
+				<xs:element name="Name" type="xs:string">
+					<xs:annotation>
+						<xs:documentation>The name of the style (extracted from Capabilities by the
+							Context document creator). This element is required.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element name="Title" type="xs:string">
+					<xs:annotation>
+						<xs:documentation>The human-readable title of the style (extracted from
+							Capabilities by the Context document creator). This element is
+						required.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element name="Abstract" type="xs:string" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>A narrative description of the current style (extracted
+							from Capabilities by the Context document creator). This element is
+							optional.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element name="LegendURL" type="owc:URLType" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>The location of an image of a map legend describing the
+							current style (extracted from Capabilities by the Context document
+							creator). This element is optional.</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+			</xs:sequence>
+			<xs:element name="SLD" type="owc:SLDType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Each Style element may alternatively define the style(s) of
+						the layer with a SLD element. The SLD element must contain required Name and
+						optional Title elements which identify the particular element of a Styled
+						Layer Descriptor to be used for this style.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:choice>
+		<xs:attribute name="current" type="xs:boolean"/>
+	</xs:complexType>
+	<xs:complexType name="SLDType">
+		<xs:sequence>
+			<xs:element name="Name" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Name of the Style</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Title" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Title of the Style</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="LegendURL" type="owc:URLType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>HTTP URL of a legend icon idicative of the
+					Style</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:choice>
+				<xs:element name="OnlineResource" type="owc:OnlineResourceType">
+					<xs:annotation>
+						<xs:documentation>HTTP URL of SLD document</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element ref="sld:StyledLayerDescriptor"/>
+				<xs:element ref="sld:FeatureTypeStyle"/>
+			</xs:choice>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="DimensionListType">
+		<xs:sequence>
+			<xs:element name="Dimension" type="owc:DimensionType" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="DimensionType">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="name" type="xs:string" use="required"/>
+				<xs:attribute name="units" type="xs:string" use="required"/>
+				<xs:attribute name="unitSymbol" type="xs:string"/>
+				<xs:attribute name="userValue" type="xs:string"/>
+				<xs:attribute name="default" type="xs:string"/>
+				<xs:attribute name="multipleValues" type="xs:boolean"/>
+				<xs:attribute name="nearestValue" type="xs:boolean"/>
+				<xs:attribute name="current" type="xs:boolean"/>
+			</xs:extension>
+		</xs:simpleContent>
+	</xs:complexType>
+	<xs:complexType name="OnlineResourceType">
+		<xs:complexContent>
+			<xs:extension base="ows:OnlineResourceType">
+				<xs:attribute name="method" type="owc:methodType" use="optional">
+					<xs:annotation>
+						<xs:documentation>Provides the name of the HTTP method to be performed on
+							this resource (either GET or POST). Note: we should use OWS Common for
+							this</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	<xs:simpleType name="methodType">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="GET"/>
+			<xs:enumeration value="POST"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:complexType name="ParameterListType">
+		<xs:annotation>
+			<xs:documentation>This allows additional elements to be added to support special
+				parameters which are not supported by this schema.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Parameter" type="owc:ExtensionType" minOccurs="0"
+				maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="ExtensionType">
+		<xs:annotation>
+			<xs:documentation>This allows additional elements to be added to support non WMS Layer
+				services.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:any namespace="##any" processContents="lax"/>
+		</xs:sequence>
+	</xs:complexType>
+</xs:schema>
diff --git ogc/ows/2.0/owsAdditionalParameters.xsd ogc/ows/2.0/owsAdditionalParameters.xsd
index 8e91265..1f534a3 100644
--- ogc/ows/2.0/owsAdditionalParameters.xsd
+++ ogc/ows/2.0/owsAdditionalParameters.xsd
@@ -53,13 +53,13 @@
   <!-- ========================================================== -->
   <complexType name="AdditionalParametersType">
     <complexContent>
-      <extension base="ows:AdditionalParametersBaseType">
+      <restriction base="ows:AdditionalParametersBaseType">
         <sequence>
           <element ref="ows:AdditionalParameter"
                    minOccurs="0"
                    maxOccurs="unbounded" />
         </sequence>
-      </extension>
+      </restriction>
     </complexContent>
   </complexType>
   <!-- ========================================================== -->
diff --git ogc/ows/2.0/owsCommon.xsd ogc/ows/2.0/owsCommon.xsd
index 0139647..9bd628e 100644
--- ogc/ows/2.0/owsCommon.xsd
+++ ogc/ows/2.0/owsCommon.xsd
@@ -71,7 +71,7 @@
     </annotation>
     <sequence>
       <element ref="ows:AbstractMetaData"
-               minOccurs="0" />
+               minOccurs="0" maxOccurs="unbounded"/>
     </sequence>
     <attributeGroup ref="xlink:simpleAttrs">
       <annotation>
diff --git ogc/sld/1.0.0/StyledLayerDescriptor.xsd ogc/sld/1.0.0/StyledLayerDescriptor.xsd
index 5a80fcd..84502fb 100644
--- ogc/sld/1.0.0/StyledLayerDescriptor.xsd
+++ ogc/sld/1.0.0/StyledLayerDescriptor.xsd
@@ -2,7 +2,7 @@
 <xsd:schema targetNamespace="http://www.opengis.net/sld" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ogc="http://www.opengis.net/ogc" xmlns:sld="http://www.opengis.net/sld" elementFormDefault="qualified" version="1.0.0.2">
 	
 	<xsd:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+	<xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 	<!-- *********************************************************************** -->
 	<xsd:annotation>
 		<xsd:documentation>
diff --git ogc/wcs/1.0.0/describeCoverage.xsd ogc/wcs/1.0.0/describeCoverage.xsd
index 0a544c4..d1ab75d 100644
--- ogc/wcs/1.0.0/describeCoverage.xsd
+++ ogc/wcs/1.0.0/describeCoverage.xsd
@@ -41,7 +41,7 @@
 	<xs:include schemaLocation="./wcsAll.xsd"/>
 	<xs:include schemaLocation="./wcsCapabilities.xsd"/>
 	<xs:include schemaLocation="./values.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.0.0/getCoverage.xsd ogc/wcs/1.0.0/getCoverage.xsd
index 94d4d17..84776f1 100644
--- ogc/wcs/1.0.0/getCoverage.xsd
+++ ogc/wcs/1.0.0/getCoverage.xsd
@@ -40,7 +40,7 @@
 	========================================================== -->
 	<xs:include schemaLocation="./wcsAll.xsd"/>
 	<xs:include schemaLocation="./describeCoverage.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.0.0/owsBase.xsd ogc/wcs/1.0.0/owsBase.xsd
index 6410894..cf9769a 100644
--- ogc/wcs/1.0.0/owsBase.xsd
+++ ogc/wcs/1.0.0/owsBase.xsd
@@ -38,7 +38,7 @@
 	<!-- ==============================================================
            includes and imports
 	============================================================== -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ============================================================
 		 Elements and types		
 	============================================================== -->
diff --git ogc/wcs/1.0.0/wcsCapabilities.xsd ogc/wcs/1.0.0/wcsCapabilities.xsd
index 5997a13..ade4e0c 100644
--- ogc/wcs/1.0.0/wcsCapabilities.xsd
+++ ogc/wcs/1.0.0/wcsCapabilities.xsd
@@ -45,7 +45,7 @@
     <!-- ========= includes and imports ================================ -->
     <xs:include schemaLocation="./wcsAll.xsd"/>
     <xs:include schemaLocation="./owsBase.xsd"/>
-    <xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+    <xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
     <xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
     <!-- ==============================================================       elements and types	============================================================== -->
     <xs:element name="GetCapabilities">
diff --git ogc/wcs/1.1/wcsCommon.xsd ogc/wcs/1.1/wcsCommon.xsd
index fe35a4c..645ae30 100644
--- ogc/wcs/1.1/wcsCommon.xsd
+++ ogc/wcs/1.1/wcsCommon.xsd
@@ -20,7 +20,7 @@ version="1.1.3">
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================		
 	elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsContents.xsd ogc/wcs/1.1/wcsContents.xsd
index d58b9e4..578de13 100644
--- ogc/wcs/1.1/wcsContents.xsd
+++ ogc/wcs/1.1/wcsContents.xsd
@@ -18,7 +18,7 @@ version="1.1.3">
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsCommon.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsCoverages.xsd ogc/wcs/1.1/wcsCoverages.xsd
index d6670f7..546a6b7 100644
--- ogc/wcs/1.1/wcsCoverages.xsd
+++ ogc/wcs/1.1/wcsCoverages.xsd
@@ -17,7 +17,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==========================================================
 		Types and elements
 	    ========================================================== -->
diff --git ogc/wcs/1.1/wcsDescribeCoverage.xsd ogc/wcs/1.1/wcsDescribeCoverage.xsd
index 29c4bfe..d026fd2 100644
--- ogc/wcs/1.1/wcsDescribeCoverage.xsd
+++ ogc/wcs/1.1/wcsDescribeCoverage.xsd
@@ -24,8 +24,8 @@ version="1.1.3">
 	<include schemaLocation="wcsCommon.xsd"></include>
 	<include schemaLocation="wcsGridCRS.xsd"></include>
 	<include schemaLocation="wcsInterpolationMethod.xsd"></include>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"></import>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"></import>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"></import>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"></import>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGetCapabilities.xsd ogc/wcs/1.1/wcsGetCapabilities.xsd
index c439f2d..043867e 100644
--- ogc/wcs/1.1/wcsGetCapabilities.xsd
+++ ogc/wcs/1.1/wcsGetCapabilities.xsd
@@ -18,7 +18,7 @@ version="1.1.3">
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsContents.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGetCoverage.xsd ogc/wcs/1.1/wcsGetCoverage.xsd
index 5cca02a..bb9eb43 100644
--- ogc/wcs/1.1/wcsGetCoverage.xsd
+++ ogc/wcs/1.1/wcsGetCoverage.xsd
@@ -20,7 +20,7 @@
 	<include schemaLocation="wcsCommon.xsd"/>
 	<include schemaLocation="wcsGridCRS.xsd"/>
 	<include schemaLocation="wcsCoverages.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================	
 		elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGridCRS.xsd ogc/wcs/1.1/wcsGridCRS.xsd
index 18d96b7..b5c6e97 100644
--- ogc/wcs/1.1/wcsGridCRS.xsd
+++ ogc/wcs/1.1/wcsGridCRS.xsd
@@ -21,7 +21,7 @@
 	<!-- ========================================================
            includes and imports
 	======================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ========================================================
 		elements and types
 	======================================================== -->
diff --git ogc/wcs/1.1/wcsInterpolationMethod.xsd ogc/wcs/1.1/wcsInterpolationMethod.xsd
index c6288d0..3a6b829 100644
--- ogc/wcs/1.1/wcsInterpolationMethod.xsd
+++ ogc/wcs/1.1/wcsInterpolationMethod.xsd
@@ -17,7 +17,7 @@ version="1.1.3">
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff --git ogc/wfs/1.0.0/WFS-basic.xsd ogc/wfs/1.0.0/WFS-basic.xsd
index 4797ba3..ddfe0c8 100644
--- ogc/wfs/1.0.0/WFS-basic.xsd
+++ ogc/wfs/1.0.0/WFS-basic.xsd
@@ -19,8 +19,8 @@
         Includes and Imports
         ============================================================== -->
    <xsd:include schemaLocation="./wfs.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/feature.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 
    <!-- ==============================================================
         REQUEST MESSAGES 
diff --git ogc/wfs/1.0.0/WFS-capabilities.xsd ogc/wfs/1.0.0/WFS-capabilities.xsd
index b408876..45c00b5 100644
--- ogc/wfs/1.0.0/WFS-capabilities.xsd
+++ ogc/wfs/1.0.0/WFS-capabilities.xsd
@@ -24,7 +24,7 @@
    <!-- ==============================================================
         Imports ...
         ============================================================== -->
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filterCapabilities.xsd" />
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filterCapabilities.xsd" />
 
    <!-- ==============================================================
         Global elements and attributes
diff --git ogc/wfs/1.0.0/WFS-transaction.xsd ogc/wfs/1.0.0/WFS-transaction.xsd
index 0cbe0c7..5072dd4 100644
--- ogc/wfs/1.0.0/WFS-transaction.xsd
+++ ogc/wfs/1.0.0/WFS-transaction.xsd
@@ -18,8 +18,8 @@
         Includes and Imports
         ============================================================== -->
    <xsd:include schemaLocation="./wfs.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/feature.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 
    <!-- ==============================================================
          REQUEST MESSAGES
diff --git ogc/wfs/1.1.0/wfs.xsd ogc/wfs/1.1.0/wfs.xsd
index 1b90d26..87ac05e 100644
--- ogc/wfs/1.1.0/wfs.xsd
+++ ogc/wfs/1.1.0/wfs.xsd
@@ -18,9 +18,9 @@
    <!-- ==============================================================
         Includes and Imports
         ============================================================== -->
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.1.0/filter.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.1.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
    <!-- ================================================================== -->
    <!-- =   BASE REQUEST TYPE                                            = -->
diff --git ogc/wfs/2.0/wfs.xsd ogc/wfs/2.0/wfs.xsd
index 299a6a0..636037a 100644
--- ogc/wfs/2.0/wfs.xsd
+++ ogc/wfs/2.0/wfs.xsd
@@ -25,9 +25,9 @@
    <xsd:import namespace="http://www.w3.org/1999/xlink"
       schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
    <xsd:import namespace="http://www.opengis.net/ows/1.1"
-      schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+      schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
    <xsd:import namespace="http://www.opengis.net/fes/2.0"
-      schemaLocation="http://schemas.opengis.net/filter/2.0/filterAll.xsd"/>
+      schemaLocation="../../filter/2.0/filterAll.xsd"/>
    <!-- ================================================================== -->
    <!-- =   BASE REQUEST TYPE                                            = -->
    <!-- ================================================================== -->
diff --git ogc/wps/1.0.0/common/DescriptionType.xsd ogc/wps/1.0.0/common/DescriptionType.xsd
index 7413109..ba87d64 100644
--- ogc/wps/1.0.0/common/DescriptionType.xsd
+++ ogc/wps/1.0.0/common/DescriptionType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/common/ProcessBriefType.xsd ogc/wps/1.0.0/common/ProcessBriefType.xsd
index e783a43..2303c3a 100644
--- ogc/wps/1.0.0/common/ProcessBriefType.xsd
+++ ogc/wps/1.0.0/common/ProcessBriefType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="DescriptionType.xsd"/>
 	<include schemaLocation="ProcessVersion.xsd"/>
 	<include schemaLocation="WSDL.xsd"/>
diff --git ogc/wps/1.0.0/common/RequestBaseType.xsd ogc/wps/1.0.0/common/RequestBaseType.xsd
index 071b0df..c98294b 100644
--- ogc/wps/1.0.0/common/RequestBaseType.xsd
+++ ogc/wps/1.0.0/common/RequestBaseType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/common/ResponseBaseType.xsd ogc/wps/1.0.0/common/ResponseBaseType.xsd
index b721bc6..0f5a95e 100644
--- ogc/wps/1.0.0/common/ResponseBaseType.xsd
+++ ogc/wps/1.0.0/common/ResponseBaseType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 	<!-- ==============================================================
 		elements and types
diff --git ogc/wps/1.0.0/wpsDescribeProcess_request.xsd ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
index 0f5e37a..2df7127 100644
--- ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
+++ ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
@@ -14,7 +14,7 @@
 	<!-- =============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/RequestBaseType.xsd"/>
 	<!-- =============================================================
diff --git ogc/wps/1.0.0/wpsDescribeProcess_response.xsd ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
index 4cdb7a7..dfb7e86 100644
--- ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
+++ ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
@@ -18,7 +18,7 @@
 	<include schemaLocation="common/ResponseBaseType.xsd"/>
 	<include schemaLocation="common/DescriptionType.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsExecute_request.xsd ogc/wps/1.0.0/wpsExecute_request.xsd
index 38f137f..b75ca69 100644
--- ogc/wps/1.0.0/wpsExecute_request.xsd
+++ ogc/wps/1.0.0/wpsExecute_request.xsd
@@ -16,7 +16,7 @@
 	============================================================== -->
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/RequestBaseType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<!-- =============================================================
 		elements and types
diff --git ogc/wps/1.0.0/wpsExecute_response.xsd ogc/wps/1.0.0/wpsExecute_response.xsd
index eade869..f407d64 100644
--- ogc/wps/1.0.0/wpsExecute_response.xsd
+++ ogc/wps/1.0.0/wpsExecute_response.xsd
@@ -18,7 +18,7 @@
 	<include schemaLocation="common/ResponseBaseType.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
 	<include schemaLocation="wpsExecute_request.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsGetCapabilities_request.xsd ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
index 718b250..c278d76 100644
--- ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
+++ ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
@@ -15,7 +15,7 @@
 		includes and imports
 	============================================================== -->
 	<include schemaLocation="wpsAll.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsGetCapabilities_response.xsd ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
index 300ba15..b2de275 100644
--- ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
+++ ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
@@ -17,7 +17,7 @@
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/WSDL.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 	<!-- ==============================================================
