diff --git ogc/filter/1.0.0/filter.xsd ogc/filter/1.0.0/filter.xsd
index f31a3b0..3a1394c 100644
--- ogc/filter/1.0.0/filter.xsd
+++ ogc/filter/1.0.0/filter.xsd
@@ -6,7 +6,7 @@
     To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<xsd:include schemaLocation="expr.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/geometry.xsd"/>
+	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/geometry.xsd"/>
 	<!-- ============================================= -->
 	<!-- FILTER EXPRESSION                             -->
 	<!-- ============================================= -->
diff --git ogc/filter/1.1.0/filter.xsd ogc/filter/1.1.0/filter.xsd
index 0eb97d2..5708362 100644
--- ogc/filter/1.1.0/filter.xsd
+++ ogc/filter/1.1.0/filter.xsd
@@ -26,7 +26,7 @@
    <xsd:include schemaLocation="filterCapabilities.xsd"/>
 
    <xsd:import namespace="http://www.opengis.net/gml"
-               schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/geometryAggregates.xsd"/>
+               schemaLocation="../../gml/3.1.1/base/geometryAggregates.xsd"/>
 
    <xsd:element name="_Id" type="ogc:AbstractIdType" abstract="true"/>
    <xsd:element name="FeatureId"
diff --git ogc/filter/2.0/filterCapabilities.xsd ogc/filter/2.0/filterCapabilities.xsd
index a7368b4..c10f712 100644
--- ogc/filter/2.0/filterCapabilities.xsd
+++ ogc/filter/2.0/filterCapabilities.xsd
@@ -23,7 +23,7 @@
                schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 
    <xsd:import namespace="http://www.opengis.net/ows/1.1"
-               schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+               schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 
    <xsd:element name="Filter_Capabilities">
       <xsd:complexType>
diff --git ogc/gml/3.1.1/base/defaultStyle.xsd ogc/gml/3.1.1/base/defaultStyle.xsd
index a5ebeb7..0a0899f 100644
--- ogc/gml/3.1.1/base/defaultStyle.xsd
+++ ogc/gml/3.1.1/base/defaultStyle.xsd
@@ -15,7 +15,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
diff --git ogc/gml/3.1.1/base/temporal.xsd ogc/gml/3.1.1/base/temporal.xsd
index 96be3f7..19ef923 100644
--- ogc/gml/3.1.1/base/temporal.xsd
+++ ogc/gml/3.1.1/base/temporal.xsd
@@ -325,6 +325,16 @@
 			<enumeration value="unknown"/>
 		</restriction>
 	</simpleType>
+	<!-- ===================================== -->
+	<simpleType name="TimeDurationType">
+		<annotation>
+			<documentation xml:lang="en">
+      Base type for describing temporal length or distance. The value space is further 
+      constrained by subtypes that conform to the ISO 8601 or ISO 11404 standards.
+      </documentation>
+		</annotation>
+		<union memberTypes="duration decimal"/>
+	</simpleType>
 	<!-- ================================================================== -->
 	<!-- ==== Convenience properties ==== -->
 	<!-- ================================================================== -->
diff --git ogc/gml/3.2.1/coordinateOperations.xsd ogc/gml/3.2.1/coordinateOperations.xsd
index 5eaacaf..7f03e51 100644
--- ogc/gml/3.2.1/coordinateOperations.xsd
+++ ogc/gml/3.2.1/coordinateOperations.xsd
@@ -19,7 +19,7 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="referenceSystems.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="../../iso/19139/20070417/gmd/gmd.xsd"/>
 	<element name="AbstractCoordinateOperation" type="gml:AbstractCoordinateOperationType" abstract="true" substitutionGroup="gml:Definition">
 		<annotation>
 			<documentation>gml:AbstractCoordinateOperation is a mathematical operation on coordinates that transforms or converts coordinates to another coordinate reference system. Many but not all coordinate operations (from CRS A to CRS B) also uniquely define the inverse operation (from CRS B to CRS A). In some cases, the operation method algorithm for the inverse operation is the same as for the forward algorithm, but the signs of some operation parameter values shall be reversed. In other cases, different algorithms are required for the forward and inverse operations, but the same operation parameter values are used. If (some) entirely different parameter values are needed, a different coordinate operation shall be defined.
diff --git ogc/gml/3.2.1/defaultStyle.xsd ogc/gml/3.2.1/defaultStyle.xsd
index 3a4635d..8419f54 100644
--- ogc/gml/3.2.1/defaultStyle.xsd
+++ ogc/gml/3.2.1/defaultStyle.xsd
@@ -13,7 +13,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../../gml/3.1.1/smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
diff --git ogc/gml/3.2.1/referenceSystems.xsd ogc/gml/3.2.1/referenceSystems.xsd
index fb7e897..2d701a8 100644
--- ogc/gml/3.2.1/referenceSystems.xsd
+++ ogc/gml/3.2.1/referenceSystems.xsd
@@ -15,7 +15,7 @@ To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	</annotation>
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="dictionary.xsd"/>
-	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="http://schemas.opengis.net/iso/19139/20070417/gmd/gmd.xsd"/>
+	<import namespace="http://www.isotc211.org/2005/gmd" schemaLocation="../../iso/19139/20070417/gmd/gmd.xsd"/>
 	<complexType name="IdentifiedObjectType" abstract="true">
 		<annotation>
 			<documentation>gml:IdentifiedObjectType provides identification properties of a CRS-related object. In gml:DefinitionType, the gml:identifier element shall be the primary name by which this object is identified, encoding the "name" attribute in the UML model.
diff --git ogc/ows/2.0/owsAdditionalParameters.xsd ogc/ows/2.0/owsAdditionalParameters.xsd
index 8e91265..1f534a3 100644
--- ogc/ows/2.0/owsAdditionalParameters.xsd
+++ ogc/ows/2.0/owsAdditionalParameters.xsd
@@ -53,13 +53,13 @@
   <!-- ========================================================== -->
   <complexType name="AdditionalParametersType">
     <complexContent>
-      <extension base="ows:AdditionalParametersBaseType">
+      <restriction base="ows:AdditionalParametersBaseType">
         <sequence>
           <element ref="ows:AdditionalParameter"
                    minOccurs="0"
                    maxOccurs="unbounded" />
         </sequence>
-      </extension>
+      </restriction>
     </complexContent>
   </complexType>
   <!-- ========================================================== -->
diff --git ogc/ows/2.0/owsCommon.xsd ogc/ows/2.0/owsCommon.xsd
index 0139647..9bd628e 100644
--- ogc/ows/2.0/owsCommon.xsd
+++ ogc/ows/2.0/owsCommon.xsd
@@ -71,7 +71,7 @@
     </annotation>
     <sequence>
       <element ref="ows:AbstractMetaData"
-               minOccurs="0" />
+               minOccurs="0" maxOccurs="unbounded"/>
     </sequence>
     <attributeGroup ref="xlink:simpleAttrs">
       <annotation>
diff --git ogc/wcs/1.0.0/describeCoverage.xsd ogc/wcs/1.0.0/describeCoverage.xsd
index 0a544c4..d1ab75d 100644
--- ogc/wcs/1.0.0/describeCoverage.xsd
+++ ogc/wcs/1.0.0/describeCoverage.xsd
@@ -41,7 +41,7 @@
 	<xs:include schemaLocation="./wcsAll.xsd"/>
 	<xs:include schemaLocation="./wcsCapabilities.xsd"/>
 	<xs:include schemaLocation="./values.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.0.0/getCoverage.xsd ogc/wcs/1.0.0/getCoverage.xsd
index 94d4d17..84776f1 100644
--- ogc/wcs/1.0.0/getCoverage.xsd
+++ ogc/wcs/1.0.0/getCoverage.xsd
@@ -40,7 +40,7 @@
 	========================================================== -->
 	<xs:include schemaLocation="./wcsAll.xsd"/>
 	<xs:include schemaLocation="./describeCoverage.xsd"/>
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.0.0/owsBase.xsd ogc/wcs/1.0.0/owsBase.xsd
index 6410894..cf9769a 100644
--- ogc/wcs/1.0.0/owsBase.xsd
+++ ogc/wcs/1.0.0/owsBase.xsd
@@ -38,7 +38,7 @@
 	<!-- ==============================================================
            includes and imports
 	============================================================== -->
-	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+	<xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ============================================================
 		 Elements and types		
 	============================================================== -->
diff --git ogc/wcs/1.0.0/wcsCapabilities.xsd ogc/wcs/1.0.0/wcsCapabilities.xsd
index 5997a13..ade4e0c 100644
--- ogc/wcs/1.0.0/wcsCapabilities.xsd
+++ ogc/wcs/1.0.0/wcsCapabilities.xsd
@@ -45,7 +45,7 @@
     <!-- ========= includes and imports ================================ -->
     <xs:include schemaLocation="./wcsAll.xsd"/>
     <xs:include schemaLocation="./owsBase.xsd"/>
-    <xs:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/wcs/1.0.0/gml4wcs.xsd"/>
+    <xs:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
     <xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
     <!-- ==============================================================       elements and types	============================================================== -->
     <xs:element name="GetCapabilities">
diff --git ogc/wcs/1.1/wcsCommon.xsd ogc/wcs/1.1/wcsCommon.xsd
index fe35a4c..645ae30 100644
--- ogc/wcs/1.1/wcsCommon.xsd
+++ ogc/wcs/1.1/wcsCommon.xsd
@@ -20,7 +20,7 @@ version="1.1.3">
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ==============================================================		
 	elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsContents.xsd ogc/wcs/1.1/wcsContents.xsd
index d58b9e4..578de13 100644
--- ogc/wcs/1.1/wcsContents.xsd
+++ ogc/wcs/1.1/wcsContents.xsd
@@ -18,7 +18,7 @@ version="1.1.3">
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsCommon.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsCoverages.xsd ogc/wcs/1.1/wcsCoverages.xsd
index d6670f7..546a6b7 100644
--- ogc/wcs/1.1/wcsCoverages.xsd
+++ ogc/wcs/1.1/wcsCoverages.xsd
@@ -17,7 +17,7 @@
 	<!-- ==============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==========================================================
 		Types and elements
 	    ========================================================== -->
diff --git ogc/wcs/1.1/wcsDescribeCoverage.xsd ogc/wcs/1.1/wcsDescribeCoverage.xsd
index 29c4bfe..d026fd2 100644
--- ogc/wcs/1.1/wcsDescribeCoverage.xsd
+++ ogc/wcs/1.1/wcsDescribeCoverage.xsd
@@ -24,8 +24,8 @@ version="1.1.3">
 	<include schemaLocation="wcsCommon.xsd"></include>
 	<include schemaLocation="wcsGridCRS.xsd"></include>
 	<include schemaLocation="wcsInterpolationMethod.xsd"></include>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"></import>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"></import>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"></import>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"></import>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGetCapabilities.xsd ogc/wcs/1.1/wcsGetCapabilities.xsd
index c439f2d..043867e 100644
--- ogc/wcs/1.1/wcsGetCapabilities.xsd
+++ ogc/wcs/1.1/wcsGetCapabilities.xsd
@@ -18,7 +18,7 @@ version="1.1.3">
        includes and imports
 	============================================================== -->
 	<include schemaLocation="wcsContents.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
        elements and types
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGetCoverage.xsd ogc/wcs/1.1/wcsGetCoverage.xsd
index 5cca02a..bb9eb43 100644
--- ogc/wcs/1.1/wcsGetCoverage.xsd
+++ ogc/wcs/1.1/wcsGetCoverage.xsd
@@ -20,7 +20,7 @@
 	<include schemaLocation="wcsCommon.xsd"/>
 	<include schemaLocation="wcsGridCRS.xsd"/>
 	<include schemaLocation="wcsCoverages.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================	
 		elements and types	
 	============================================================== -->
diff --git ogc/wcs/1.1/wcsGridCRS.xsd ogc/wcs/1.1/wcsGridCRS.xsd
index 18d96b7..b5c6e97 100644
--- ogc/wcs/1.1/wcsGridCRS.xsd
+++ ogc/wcs/1.1/wcsGridCRS.xsd
@@ -21,7 +21,7 @@
 	<!-- ========================================================
            includes and imports
 	======================================================== -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
+	<import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
 	<!-- ========================================================
 		elements and types
 	======================================================== -->
diff --git ogc/wcs/1.1/wcsInterpolationMethod.xsd ogc/wcs/1.1/wcsInterpolationMethod.xsd
index c6288d0..3a6b829 100644
--- ogc/wcs/1.1/wcsInterpolationMethod.xsd
+++ ogc/wcs/1.1/wcsInterpolationMethod.xsd
@@ -17,7 +17,7 @@ version="1.1.3">
 	<!-- ==============================================================
  	includes and imports	
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================    
 	   elements and types	
 	============================================================== -->
diff --git ogc/wfs/1.0.0/WFS-basic.xsd ogc/wfs/1.0.0/WFS-basic.xsd
index 4797ba3..ddfe0c8 100644
--- ogc/wfs/1.0.0/WFS-basic.xsd
+++ ogc/wfs/1.0.0/WFS-basic.xsd
@@ -19,8 +19,8 @@
         Includes and Imports
         ============================================================== -->
    <xsd:include schemaLocation="./wfs.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/feature.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 
    <!-- ==============================================================
         REQUEST MESSAGES 
diff --git ogc/wfs/1.0.0/WFS-capabilities.xsd ogc/wfs/1.0.0/WFS-capabilities.xsd
index b408876..45c00b5 100644
--- ogc/wfs/1.0.0/WFS-capabilities.xsd
+++ ogc/wfs/1.0.0/WFS-capabilities.xsd
@@ -24,7 +24,7 @@
    <!-- ==============================================================
         Imports ...
         ============================================================== -->
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filterCapabilities.xsd" />
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filterCapabilities.xsd" />
 
    <!-- ==============================================================
         Global elements and attributes
diff --git ogc/wfs/1.0.0/WFS-transaction.xsd ogc/wfs/1.0.0/WFS-transaction.xsd
index 0cbe0c7..5072dd4 100644
--- ogc/wfs/1.0.0/WFS-transaction.xsd
+++ ogc/wfs/1.0.0/WFS-transaction.xsd
@@ -18,8 +18,8 @@
         Includes and Imports
         ============================================================== -->
    <xsd:include schemaLocation="./wfs.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/2.1.2/feature.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.0.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/2.1.2/feature.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.0.0/filter.xsd"/>
 
    <!-- ==============================================================
          REQUEST MESSAGES
diff --git ogc/wfs/1.1.0/wfs.xsd ogc/wfs/1.1.0/wfs.xsd
index 1b90d26..87ac05e 100644
--- ogc/wfs/1.1.0/wfs.xsd
+++ ogc/wfs/1.1.0/wfs.xsd
@@ -18,9 +18,9 @@
    <!-- ==============================================================
         Includes and Imports
         ============================================================== -->
-   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="http://schemas.opengis.net/filter/1.1.0/filter.xsd"/>
-   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="http://schemas.opengis.net/ows/1.0.0/owsAll.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/gml" schemaLocation="../../gml/3.1.1/base/gml.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ogc" schemaLocation="../../filter/1.1.0/filter.xsd"/>
+   <xsd:import namespace="http://www.opengis.net/ows" schemaLocation="../../ows/1.0.0/owsAll.xsd"/>
 
    <!-- ================================================================== -->
    <!-- =   BASE REQUEST TYPE                                            = -->
diff --git ogc/wfs/2.0/wfs.xsd ogc/wfs/2.0/wfs.xsd
index 299a6a0..636037a 100644
--- ogc/wfs/2.0/wfs.xsd
+++ ogc/wfs/2.0/wfs.xsd
@@ -25,9 +25,9 @@
    <xsd:import namespace="http://www.w3.org/1999/xlink"
       schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
    <xsd:import namespace="http://www.opengis.net/ows/1.1"
-      schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+      schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
    <xsd:import namespace="http://www.opengis.net/fes/2.0"
-      schemaLocation="http://schemas.opengis.net/filter/2.0/filterAll.xsd"/>
+      schemaLocation="../../filter/2.0/filterAll.xsd"/>
    <!-- ================================================================== -->
    <!-- =   BASE REQUEST TYPE                                            = -->
    <!-- ================================================================== -->
diff --git ogc/wps/1.0.0/common/DescriptionType.xsd ogc/wps/1.0.0/common/DescriptionType.xsd
index 7413109..ba87d64 100644
--- ogc/wps/1.0.0/common/DescriptionType.xsd
+++ ogc/wps/1.0.0/common/DescriptionType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/common/ProcessBriefType.xsd ogc/wps/1.0.0/common/ProcessBriefType.xsd
index e783a43..2303c3a 100644
--- ogc/wps/1.0.0/common/ProcessBriefType.xsd
+++ ogc/wps/1.0.0/common/ProcessBriefType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="DescriptionType.xsd"/>
 	<include schemaLocation="ProcessVersion.xsd"/>
 	<include schemaLocation="WSDL.xsd"/>
diff --git ogc/wps/1.0.0/common/RequestBaseType.xsd ogc/wps/1.0.0/common/RequestBaseType.xsd
index 071b0df..c98294b 100644
--- ogc/wps/1.0.0/common/RequestBaseType.xsd
+++ ogc/wps/1.0.0/common/RequestBaseType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<!-- ==============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/common/ResponseBaseType.xsd ogc/wps/1.0.0/common/ResponseBaseType.xsd
index b721bc6..0f5a95e 100644
--- ogc/wps/1.0.0/common/ResponseBaseType.xsd
+++ ogc/wps/1.0.0/common/ResponseBaseType.xsd
@@ -14,7 +14,7 @@
 	<!-- ==============================================================
 		Import or include schema
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 	<!-- ==============================================================
 		elements and types
diff --git ogc/wps/1.0.0/wpsDescribeProcess_request.xsd ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
index 0f5e37a..2df7127 100644
--- ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
+++ ogc/wps/1.0.0/wpsDescribeProcess_request.xsd
@@ -14,7 +14,7 @@
 	<!-- =============================================================
 		includes and imports
 	============================================================== -->
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/RequestBaseType.xsd"/>
 	<!-- =============================================================
diff --git ogc/wps/1.0.0/wpsDescribeProcess_response.xsd ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
index 4cdb7a7..dfb7e86 100644
--- ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
+++ ogc/wps/1.0.0/wpsDescribeProcess_response.xsd
@@ -18,7 +18,7 @@
 	<include schemaLocation="common/ResponseBaseType.xsd"/>
 	<include schemaLocation="common/DescriptionType.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsExecute_request.xsd ogc/wps/1.0.0/wpsExecute_request.xsd
index 38f137f..b75ca69 100644
--- ogc/wps/1.0.0/wpsExecute_request.xsd
+++ ogc/wps/1.0.0/wpsExecute_request.xsd
@@ -16,7 +16,7 @@
 	============================================================== -->
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/RequestBaseType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<!-- =============================================================
 		elements and types
diff --git ogc/wps/1.0.0/wpsExecute_response.xsd ogc/wps/1.0.0/wpsExecute_response.xsd
index eade869..f407d64 100644
--- ogc/wps/1.0.0/wpsExecute_response.xsd
+++ ogc/wps/1.0.0/wpsExecute_response.xsd
@@ -18,7 +18,7 @@
 	<include schemaLocation="common/ResponseBaseType.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
 	<include schemaLocation="wpsExecute_request.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsGetCapabilities_request.xsd ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
index 718b250..c278d76 100644
--- ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
+++ ogc/wps/1.0.0/wpsGetCapabilities_request.xsd
@@ -15,7 +15,7 @@
 		includes and imports
 	============================================================== -->
 	<include schemaLocation="wpsAll.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<!-- =============================================================
 		elements and types
 	============================================================== -->
diff --git ogc/wps/1.0.0/wpsGetCapabilities_response.xsd ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
index 300ba15..b2de275 100644
--- ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
+++ ogc/wps/1.0.0/wpsGetCapabilities_response.xsd
@@ -17,7 +17,7 @@
 	<include schemaLocation="wpsAll.xsd"/>
 	<include schemaLocation="common/WSDL.xsd"/>
 	<include schemaLocation="common/ProcessBriefType.xsd"/>
-	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="http://schemas.opengis.net/ows/1.1.0/owsAll.xsd"/>
+	<import namespace="http://www.opengis.net/ows/1.1" schemaLocation="../../ows/1.1.0/owsAll.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd"/>
 	<!-- ==============================================================
