diff -urN ../resources-original/ogc/context/1.0.0/collection.xsd ogc/context/1.0.0/collection.xsd
--- ../resources-original/ogc/context/1.0.0/collection.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/context/1.0.0/collection.xsd	Wed Nov 11 19:51:12 2015
@@ -8,6 +8,7 @@
 -->
 <xs:schema targetNamespace="http://www.opengis.net/context" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:context="http://www.opengis.net/context" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" version="1.0.0.2">
 	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
+	<xs:include schemaLocation="context.xsd"/>
 	<xs:element name="ViewContextCollection" type="context:ViewContextCollectionType"/>
 	<xs:complexType name="ViewContextCollectionType">
 		<xs:sequence>
@@ -28,7 +29,9 @@
 			<xs:element name="OnlineResource" type="context:OnlineResourceType"/>
 		</xs:sequence>
 	</xs:complexType>
+	<!--
 	<xs:complexType name="OnlineResourceType">
 		<xs:attributeGroup ref="xlink:simpleAttrs"/>
 	</xs:complexType>
+	-->
 </xs:schema>
diff -urN ../resources-original/ogc/filter/1.0.0/expr.xsd ogc/filter/1.0.0/expr.xsd
--- ../resources-original/ogc/filter/1.0.0/expr.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/filter/1.0.0/expr.xsd	Wed Nov 11 19:51:12 2015
@@ -68,7 +68,11 @@
   </xsd:complexType>
   <xsd:complexType name="PropertyNameType">
     <xsd:complexContent mixed="true">
-      <xsd:extension base="ogc:ExpressionType"/>
+      <xsd:extension base="ogc:ExpressionType">
+        <xsd:sequence>
+          <xsd:any minOccurs="0"/>
+        </xsd:sequence>
+      </xsd:extension>
     </xsd:complexContent>
   </xsd:complexType>
 </xsd:schema>
diff -urN ../resources-original/ogc/filter/1.1.0/expr.xsd ogc/filter/1.1.0/expr.xsd
--- ../resources-original/ogc/filter/1.1.0/expr.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/filter/1.1.0/expr.xsd	Wed Nov 11 19:51:12 2015
@@ -61,7 +61,11 @@
    </xsd:complexType>
    <xsd:complexType name="PropertyNameType">
       <xsd:complexContent mixed="true">
-         <xsd:extension base="ogc:ExpressionType"/>
+        <xsd:extension base="ogc:ExpressionType">
+          <xsd:sequence>
+            <xsd:any minOccurs="0"/>
+          </xsd:sequence>
+        </xsd:extension>
       </xsd:complexContent>
    </xsd:complexType>
 </xsd:schema>
diff -urN ../resources-original/ogc/indoorgml/1.0/indoorgmlnavi.xsd ogc/indoorgml/1.0/indoorgmlnavi.xsd
--- ../resources-original/ogc/indoorgml/1.0/indoorgmlnavi.xsd	Wed Dec  3 15:21:26 2014
+++ ogc/indoorgml/1.0/indoorgmlnavi.xsd	Wed Nov 11 19:51:12 2015
@@ -8,7 +8,7 @@
 		</xs:documentation>
 	</xs:annotation>
 	<xs:import namespace="http://www.opengis.net/gml/3.2" schemaLocation="http://schemas.opengis.net/gml/3.2.1/gml.xsd"/>
-	<xs:import namespace="http://www.opengis.net/indoorgml/1.0/core" schemaLocation="http://schemas.opengis.net/indoorgml/1.0/core/indoorgmlcore.xsd"/>
+	<xs:import namespace="http://www.opengis.net/indoorgml/1.0/core" schemaLocation="http://schemas.opengis.net/indoorgml/1.0/indoorgmlcore.xsd"/>
 	<!-- ====================================================================== -->
 	<xs:element name="NavigableSpace" type="NavigableSpaceType" substitutionGroup="IndoorCore:CellSpace">
 		<xs:annotation>
diff -urN ../resources-original/ogc/kml/2.3/ogckml23.xsd ogc/kml/2.3/ogckml23.xsd
--- ../resources-original/ogc/kml/2.3/ogckml23.xsd	Thu Oct 22 10:14:10 2015
+++ ogc/kml/2.3/ogckml23.xsd	Wed Nov 11 19:54:26 2015
@@ -23,9 +23,9 @@
   <import namespace="urn:oasis:names:tc:ciq:xsdschema:xAL:2.0"
     schemaLocation="http://docs.oasis-open.org/election/external/xAL.xsd"/>
 
-  <defaultOpenContent mode="interleave">
+  <!--defaultOpenContent mode="interleave">
     <any namespace="##other" processContents="lax"/>
-  </defaultOpenContent>
+  </defaultOpenContent-->
 
   <!-- KML field types (simple content) -->
 
@@ -767,167 +767,167 @@
     <attribute name="version" type="kml:kmlVersionType" default="2.2"/>
     <anyAttribute namespace="##other" processContents="lax"/>
     <!-- TODO(sean): Make sure these are all the parent elements we care about. -->
-    <assert test="if (.//kml:Track)
+    <!-- assert test="if (.//kml:Track)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:Track is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:MultiTrack)
+    </assert-->
+    <!-- assert test="if (.//kml:MultiTrack)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:MultiTrack is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:Tour)
+    </assert-->
+    <!-- assert test="if (.//kml:Tour)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:Tour is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:LatLonQuad)
+    </assert-->
+    <!-- assert test="if (.//kml:LatLonQuad)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:LatLonQuad is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:seaFloorAltitudeMode)
+    </assert-->
+    <!-- assert test="if (.//kml:seaFloorAltitudeMode)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:seaFloorAltitudeMode is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:altitudeOffset)
+    </assert-->
+    <!-- assert test="if (.//kml:altitudeOffset)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:altitudeOffset is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:balloonVisibility)
+    </assert-->
+    <!-- assert test="if (.//kml:balloonVisibility)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:balloonVisibility is not allowed in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:Camera/kml:TimeStamp)
+    </assert-->
+    <!-- assert test="if (.//kml:Camera/kml:TimeStamp)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:TimeStamp is not a valid child of kml:Camera in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:Camera/kml:TimeSpan)
+    </assert-->
+    <!-- assert test="if (.//kml:Camera/kml:TimeSpan)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:TimeSpan is not a valid child of kml:Camera in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:LookAt/kml:TimeStamp)
+    </assert-->
+    <!-- assert test="if (.//kml:LookAt/kml:TimeStamp)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:TimeStamp is not a valid child of kml:LookAt in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-    <assert test="if (.//kml:LookAt/kml:TimeSpan)
+    </assert-->
+    <!-- assert test="if (.//kml:LookAt/kml:TimeSpan)
       then starts-with(@version,'2.3')
       else true()">
       <annotation>
         <documentation>kml:LookAt is not a valid child of kml:LookAt in versions of KML prior to 2.3.0.</documentation>
       </annotation>
-    </assert>
-   <assert test="if (.//kml:SimpleArrayField)
+    </assert-->
+   <!-- assert test="if (.//kml:SimpleArrayField)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:SimpleArrayField is not allowed in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:SimpleField/@uom)
+   </assert-->   
+   <!-- assert test="if (.//kml:SimpleField/@uom)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>Attribute 'uom' is not a valid child of kml:SimpleField in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Data/@uom)
+   </assert-->   
+   <!-- assert test="if (.//kml:Data/@uom)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>Attribute 'uom' is not a valid child of kml:Data in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Camera/kml:horizFov)
+   </assert-->   
+   <!-- assert test="if (.//kml:Camera/kml:horizFov)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:horizFov is not a valid child of kml:Camera in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:LookAt/kml:horizFov)
+   </assert-->   
+   <!-- assert test="if (.//kml:LookAt/kml:horizFov)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:horizFov is not a valid child of kml:LookAt in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Create/kml:MultiGeometry)
+   </assert-->   
+   <!-- assert test="if (.//kml:Create/kml:MultiGeometry)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:MultiGeometry is not a valid child of kml:Create in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:MultiGeometry)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:MultiGeometry)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:MultiGeometry is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:Point)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:Point)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:Point is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:LineString)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:LineString)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:LineString is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:LinearRing)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:LinearRing)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:LinearRing is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:Polygon)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:Polygon)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:Polygon is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
-   <assert test="if (.//kml:Delete/kml:Model)
+   </assert-->   
+   <!-- assert test="if (.//kml:Delete/kml:Model)
     then starts-with(@version,'2.3')
     else true()">
     <annotation>
      <documentation>kml:Model is not a valid child of kml:Delete in versions of KML prior to 2.3.0.</documentation>
     </annotation>
-   </assert>   
+   </assert-->   
   </complexType>
   <element name="KmlSimpleExtensionGroup" abstract="true" type="anySimpleType"/>
   <element name="KmlObjectExtensionGroup" abstract="true"
@@ -1105,16 +1105,16 @@
           <element ref="kml:RegionObjectExtensionGroup" minOccurs="0"
             maxOccurs="unbounded"/>
         </all>
-        <assert test="not(./kml:LatLonBox) ">
+        <!-- assert test="not(./kml:LatLonBox) ">
           <annotation>
             <documentation>kml:LatLonBox is prohibited in kml:Region. Use kml:LatLonAltBox instead.</documentation>
           </annotation>
-        </assert>
-        <assert test="not(./kml:LatLonQuad) ">
+        </assert-->
+        <!-- assert test="not(./kml:LatLonQuad) ">
           <annotation>
             <documentation>kml:LatLonQuad is prohibited in kml:Region. Use kml:LatLonAltBox instead.</documentation>
           </annotation>
-        </assert>
+        </assert-->
       </extension>
     </complexContent>
   </complexType>
@@ -1378,16 +1378,16 @@
           <element ref="kml:TrackObjectExtensionGroup" minOccurs="0"
             maxOccurs="unbounded"/>
         </all>
-        <assert test="count(./kml:when) = count(./kml:coord)">
+        <!-- assert test="count(./kml:when) = count(./kml:coord)">
           <annotation>
             <documentation>kml:Track must have an equal number of kml:when and kml:coord elements.</documentation>
           </annotation>
-        </assert>
-        <assert test="if(./kml:angles) then count(./kml:angles) = count(./kml:when) else true()">
+        </assert-->
+        <!-- assert test="if(./kml:angles) then count(./kml:angles) = count(./kml:when) else true()">
           <annotation>
             <documentation>If kml:angles appears in kml:Track there must be an equal number of kml:angles and kml:when elements.</documentation>
           </annotation>
-        </assert>
+        </assert-->
       </extension>
     </complexContent>
   </complexType>
@@ -1540,11 +1540,11 @@
           <element ref="kml:GroundOverlayObjectExtensionGroup" minOccurs="0"
             maxOccurs="unbounded"/>
         </all>
-        <assert test="not(./kml:LatLonAltBox)">
+        <!-- assert test="not(./kml:LatLonAltBox)">
           <annotation>
             <documentation>kml:LatLonAltBox is prohibited in kml:GroundOverlay. Use kml:LatLonBox or kml:LatLonQuad instead.</documentation>
           </annotation>
-        </assert>
+        </assert-->
       </extension>
     </complexContent>
   </complexType>
diff -urN ../resources-original/ogc/ols/1.1.0/ADT.xsd ogc/ols/1.1.0/ADT.xsd
--- ../resources-original/ogc/ols/1.1.0/ADT.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/ADT.xsd	Wed Nov 11 19:51:12 2015
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="geometry.xsd"/>
 	<!--  The ADTs defined in this document represent the minimum set of essential elements as determined from the analysis of the APIs provided by OpenLS sponsors and participants. There are many additional elements that may be added but these are considered to be optional. The APIs considered include: Oracle, Webraska, ESRI, Vodafone, opt[e]way, NavTech, IntelliWhere, and LocatioNet. -->
 	<!--=============================================================================
diff -urN ../resources-original/ogc/ols/1.1.0/DirectoryService.xsd ogc/ols/1.1.0/DirectoryService.xsd
--- ../resources-original/ogc/ols/1.1.0/DirectoryService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/DirectoryService.xsd	Wed Nov 11 19:51:12 2015
@@ -5,7 +5,8 @@
  Copyright (c) 2006,2010 Open Geospatial Consortium.
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="DirectoryRequest" type="xls:DirectoryRequestType" substitutionGroup="xls:_RequestParameters"/>
diff -urN ../resources-original/ogc/ols/1.1.0/GatewayService.xsd ogc/ols/1.1.0/GatewayService.xsd
--- ../resources-original/ogc/ols/1.1.0/GatewayService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/GatewayService.xsd	Wed Nov 11 19:51:12 2015
@@ -5,7 +5,8 @@
  Copyright (c) 2006,2010 Open Geospatial Consortium.
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!--================= Gateway Request and Response Context=========================-->
 	<element name="_GatewayParameters" type="xls:AbstractGatewayParametersType" abstract="false">
diff -urN ../resources-original/ogc/ols/1.1.0/LocationUtilityService.xsd ogc/ols/1.1.0/LocationUtilityService.xsd
--- ../resources-original/ogc/ols/1.1.0/LocationUtilityService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/LocationUtilityService.xsd	Wed Nov 11 19:51:12 2015
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="GeocodeRequest" type="xls:GeocodeRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.1.0/PresentationService.xsd ogc/ols/1.1.0/PresentationService.xsd
--- ../resources-original/ogc/ols/1.1.0/PresentationService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/PresentationService.xsd	Wed Nov 11 19:51:12 2015
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ==============================================GetCapabilities Request=====================-->
 	<element name="GetPortrayMapCapabilitiesRequest" type="xls:GetPortrayMapCapabilitiesRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.1.0/RouteService.xsd ogc/ols/1.1.0/RouteService.xsd
--- ../resources-original/ogc/ols/1.1.0/RouteService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/RouteService.xsd	Wed Nov 11 19:51:12 2015
@@ -7,7 +7,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ======================================================================================
 				Route Service Basic
diff -urN ../resources-original/ogc/ols/1.1.0/geometry.xsd ogc/ols/1.1.0/geometry.xsd
--- ../resources-original/ogc/ols/1.1.0/geometry.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.1.0/geometry.xsd	Wed Nov 11 19:51:12 2015
@@ -6,7 +6,8 @@
  To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 -->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/>
 	<include schemaLocation="UOM.xsd"/>
 	<!-- =============================== type ===================================== -->
diff -urN ../resources-original/ogc/ols/1.2/ADT.xsd ogc/ols/1.2/ADT.xsd
--- ../resources-original/ogc/ols/1.2/ADT.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/ADT.xsd	Wed Nov 11 19:51:12 2015
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="geometry.xsd"/>
 	<!--  The ADTs defined in this document represent the minimum set of essential elements as determined from the analysis of the APIs provided by OpenLS sponsors and participants. There are many additional elements that may be added but these are considered to be optional. The APIs considered include: Oracle, Webraska, ESRI, Vodafone, opt[e]way, NavTech, IntelliWhere, and LocatioNet. -->
 	<!--=============================================================================
diff -urN ../resources-original/ogc/ols/1.2/DirectoryService.xsd ogc/ols/1.2/DirectoryService.xsd
--- ../resources-original/ogc/ols/1.2/DirectoryService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/DirectoryService.xsd	Wed Nov 11 19:51:12 2015
@@ -9,7 +9,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="DirectoryRequest" type="xls:DirectoryRequestType" substitutionGroup="xls:_RequestParameters"/>
diff -urN ../resources-original/ogc/ols/1.2/GatewayService.xsd ogc/ols/1.2/GatewayService.xsd
--- ../resources-original/ogc/ols/1.2/GatewayService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/GatewayService.xsd	Wed Nov 11 19:51:12 2015
@@ -9,7 +9,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!--================= Gateway Request and Response Context=========================-->
 	<element name="_GatewayParameters" type="xls:AbstractGatewayParametersType" abstract="false">
diff -urN ../resources-original/ogc/ols/1.2/LocationUtilityService.xsd ogc/ols/1.2/LocationUtilityService.xsd
--- ../resources-original/ogc/ols/1.2/LocationUtilityService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/LocationUtilityService.xsd	Wed Nov 11 19:51:12 2015
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ====================================================== -->
 	<element name="GeocodeRequest" type="xls:GeocodeRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.2/PresentationService.xsd ogc/ols/1.2/PresentationService.xsd
--- ../resources-original/ogc/ols/1.2/PresentationService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/PresentationService.xsd	Wed Nov 11 19:51:12 2015
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ==============================================GetCapabilities Request=====================-->
 	<element name="GetPortrayMapCapabilitiesRequest" type="xls:GetPortrayMapCapabilitiesRequestType" substitutionGroup="xls:_RequestParameters">
diff -urN ../resources-original/ogc/ols/1.2/RouteService.xsd ogc/ols/1.2/RouteService.xsd
--- ../resources-original/ogc/ols/1.2/RouteService.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/RouteService.xsd	Wed Nov 11 19:51:12 2015
@@ -10,7 +10,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<include schemaLocation="XLS.xsd"/>
 	<!-- ======================================================================================
 				Route Service Basic
diff -urN ../resources-original/ogc/ols/1.2/geometry.xsd ogc/ols/1.2/geometry.xsd
--- ../resources-original/ogc/ols/1.2/geometry.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ols/1.2/geometry.xsd	Wed Nov 11 19:51:12 2015
@@ -11,7 +11,8 @@
 		To obtain additional rights of use, visit http://www.opengeospatial.org/legal/ .
 	-->
 	<include schemaLocation="olsAll.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/> 
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.w3.org/1999/xlink.xsd"/> 
 	<include schemaLocation="UOM.xsd"/> 
 	<!-- =============================== type ===================================== --> 
diff -urN ../resources-original/ogc/ols/1.3/nav-Positioning.xsd ogc/ols/1.3/nav-Positioning.xsd
--- ../resources-original/ogc/ols/1.3/nav-Positioning.xsd	Thu Jul 31 14:46:16 2014
+++ ogc/ols/1.3/nav-Positioning.xsd	Wed Nov 11 19:51:12 2015
@@ -8,7 +8,8 @@
 		</xsd:documentation>
 	</xsd:annotation>
 	<xsd:include schemaLocation="nav-All.xsd"/>
-	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/>
+	<!--xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/-->
+	<xsd:import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<xsd:import namespace="http://www.opengis.net/xls" schemaLocation="http://schemas.opengis.net/ols/1.2/XLS.xsd"/>
 	<xsd:include schemaLocation="nav-CursorUtil.xsd"/>
 	<xsd:include schemaLocation="nav-UOM.xsd"/>
diff -urN ../resources-original/ogc/ols/1.3/nav-RouteService.xsd ogc/ols/1.3/nav-RouteService.xsd
--- ../resources-original/ogc/ols/1.3/nav-RouteService.xsd	Thu Jul 31 14:46:16 2014
+++ ogc/ols/1.3/nav-RouteService.xsd	Wed Nov 11 19:51:12 2015
@@ -8,7 +8,8 @@
 		</documentation>
 	</annotation>
 	<include schemaLocation="nav-All.xsd"/>
-	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/>
+	<!--import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/ols/1.2/gml4xls.xsd"/-->
+	<import namespace="http://www.opengis.net/gml" schemaLocation="http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"/>
 	<import namespace="http://www.opengis.net/xls" schemaLocation="http://schemas.opengis.net/ols/1.2/XLS.xsd"/>
 	<import namespace="http://www.opengis.net/xls" schemaLocation="http://schemas.opengis.net/ols/1.2/RouteService.xsd"/>
 	<include schemaLocation="nav-Positioning.xsd"/>
diff -urN ../resources-original/ogc/ows/1.1.0/owsContents.xsd ogc/ows/1.1.0/owsContents.xsd
--- ../resources-original/ogc/ows/1.1.0/owsContents.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ows/1.1.0/owsContents.xsd	Wed Nov 11 19:51:12 2015
@@ -84,4 +84,8 @@
 		</complexContent>
 	</complexType>
 	<!-- ===========================================================-->
+	
+	<!-- This is to force JAXB generate reference properties -->
+	<element name="DatasetDescriptionSummaryExtension" type="ows:DatasetDescriptionSummaryBaseType" substitutionGroup="ows:DatasetDescriptionSummary"/>
+	<element name="OtherSourceExtension" type="ows:MetadataType" substitutionGroup="ows:OtherSource"/>
 </schema>
diff -urN ../resources-original/ogc/ows/2.0/owsAdditionalParameters.xsd ogc/ows/2.0/owsAdditionalParameters.xsd
--- ../resources-original/ogc/ows/2.0/owsAdditionalParameters.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/ows/2.0/owsAdditionalParameters.xsd	Wed Nov 11 19:51:12 2015
@@ -53,13 +53,13 @@
   <!-- ========================================================== -->
   <complexType name="AdditionalParametersType">
     <complexContent>
-      <extension base="ows:AdditionalParametersBaseType">
+      <restriction base="ows:AdditionalParametersBaseType">
         <sequence>
           <element ref="ows:AdditionalParameter"
                    minOccurs="0"
                    maxOccurs="unbounded" />
         </sequence>
-      </extension>
+      </restriction>
     </complexContent>
   </complexType>
   <!-- ========================================================== -->
diff -urN ../resources-original/ogc/sweCommon/1.0.0/aggregateTypes.xsd ogc/sweCommon/1.0.0/aggregateTypes.xsd
--- ../resources-original/ogc/sweCommon/1.0.0/aggregateTypes.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/sweCommon/1.0.0/aggregateTypes.xsd	Wed Nov 11 19:51:12 2015
@@ -139,7 +139,7 @@
 		<xs:annotation>
 			<xs:documentation>Use to point or include data values inline</xs:documentation>
 		</xs:annotation>
-		<xs:complexContent>
+		<xs:complexContent mixed="true">
 			<xs:extension base="xs:anyType">
 				<xs:attributeGroup ref="gml:AssociationAttributeGroup"/>
 			</xs:extension>
diff -urN ../resources-original/ogc/sweCommon/1.0.1/aggregateTypes.xsd ogc/sweCommon/1.0.1/aggregateTypes.xsd
--- ../resources-original/ogc/sweCommon/1.0.1/aggregateTypes.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/sweCommon/1.0.1/aggregateTypes.xsd	Wed Nov 11 19:51:12 2015
@@ -138,7 +138,7 @@
 		<xs:annotation>
 			<xs:documentation>Use to point or include data values inline</xs:documentation>
 		</xs:annotation>
-		<xs:complexContent>
+		<xs:complexContent mixed="true">
 			<xs:extension base="xs:anyType">
 				<xs:attribute name="recordCount" type="xs:positiveInteger"/>
 				<xs:attributeGroup ref="gml:AssociationAttributeGroup"/>
diff -urN ../resources-original/ogc/sweCommon/2.0/basic_types.xsd ogc/sweCommon/2.0/basic_types.xsd
--- ../resources-original/ogc/sweCommon/2.0/basic_types.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/sweCommon/2.0/basic_types.xsd	Wed Nov 11 19:51:12 2015
@@ -78,7 +78,7 @@
 	</complexType>
 	<!-- ================================================= -->
 	<complexType name="EncodedValuesPropertyType">
-		<complexContent>
+		<complexContent mixed="true">
 			<extension base="anyType">
 			  <attributeGroup ref="swe:AssociationAttributeGroup"/>
 			</extension>
diff -urN ../resources-original/ogc/wcs/1.0.0/wcsCapabilities.xsd ogc/wcs/1.0.0/wcsCapabilities.xsd
--- ../resources-original/ogc/wcs/1.0.0/wcsCapabilities.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/wcs/1.0.0/wcsCapabilities.xsd	Wed Nov 11 19:51:12 2015
@@ -444,9 +444,9 @@
         </xs:annotation>
         <xs:complexContent>
             <xs:restriction base="gml:AbstractGMLType">
-                <xs:sequence>
+                <!--xs:sequence>
                     <xs:element ref="metadataLink" minOccurs="0" maxOccurs="unbounded"/>
-                </xs:sequence>
+                </xs:sequence-->
             </xs:restriction>
         </xs:complexContent>
     </xs:complexType>
diff -urN ../resources-original/ogc/wms/1.3.0/capabilities_1_3_0.xsd ogc/wms/1.3.0/capabilities_1_3_0.xsd
--- ../resources-original/ogc/wms/1.3.0/capabilities_1_3_0.xsd	Sat Jul 21 21:59:38 2012
+++ ogc/wms/1.3.0/capabilities_1_3_0.xsd	Wed Nov 11 19:51:12 2015
@@ -615,4 +615,7 @@
 			<maxInclusive value="90"/>
 		</restriction>
 	</simpleType>
+
+	<element name="OtherExtendedCapabilities" substitutionGroup="wms:_ExtendedCapabilities"/>
+	<element name="OtherExtendedOperation" type="wms:OperationType" substitutionGroup="wms:_ExtendedOperation"/>
 </schema>
