diff --git ogc/gml/3.1.1/base/defaultStyle.xsd ogc/gml/3.1.1/base/defaultStyle.xsd
index a5ebeb7..0a0899f 100644
--- ogc/gml/3.1.1/base/defaultStyle.xsd
+++ ogc/gml/3.1.1/base/defaultStyle.xsd
@@ -15,7 +15,7 @@
 	============================================================== -->
 	<include schemaLocation="gml.xsd"/>
 	<include schemaLocation="measures.xsd"/>
-	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="http://schemas.opengis.net/gml/3.1.1/smil/smil20.xsd"/>
+	<import namespace="http://www.w3.org/2001/SMIL20/" schemaLocation="../smil/smil20.xsd"/>
 	<!-- ==============================================================
       the Style property
 	============================================================== -->
